<!--
The {#rmd-basics} text after the chapter declaration will allow us to link throughout the document back to the beginning of Chapter 1.  These labels will automatically be generated (if not specified) by changing the spaces to hyphens and capital letters to lowercase.  Look for the reference to this label at the beginning of Chapter 2.
-->

# Flow regulation associated with decreased genetic health of a river-breeding frog species {#reg-health}

## Introduction

Rivers simultaneously connect and carve the landscapes through which they flow. Rivers provide corridors of connectivity for riparian and aquatic organisms such as fish, amphibians, and macroinvertebrates [@pringle_what_2003; @wiens_riverine_2002], while also acting as physical barriers on the landscape for many terrestrial organisms [@caze_could_2016; @voelker_river_2013]. Hydrologic connectivity [@pringle_what_2003] transfers energy, organisms and ultimately genetic variation and thus is a critical component for population persistence in dynamic systems where populations must constantly adapt to temporal and spatial changes. In Mediterranean climates, rivers have strong seasonal patterns associated with cold, wet winters and warm, dry summers. Native aquatic organisms have evolved life histories well adapted to these natural patterns, which are both predictable and seasonal [@tonkin_seasonality_2017; @yarnell_ecology_2010].

River regulation, or the hydrological alteration of flow by dams and diversions, impacts the seasonal and interannual flow variability within a watershed. Regulation changes the natural flow regime and dramatically alters geomorphic and hydrologic connectivity of watersheds [@poff_homogenization_2007], which may restrict natural population connectivity [@schick_directed_2007; @shaw_importance_2016]. River regulation can change flow frequency, magnitude, duration, timing, and rate of change, which can have significant impacts on aquatic organisms and ecological processes [@poff_homogenization_2007; @yarnell_ecology_2010]. River regulation, and more specifically, regulation associated with hydropower generation, has been implicated as a cause of fundamental changes to downstream aquatic ecosystems [@bunn_basic_2002; @moyle_rapid_2011; @power_dams_1996]. The hydrological regimes of over half of the world‚Äôs largest rivers have been altered by large dams [@nilsson_fragmentation_2005] and only recently has the extent of flow alteration and the associated ecosystem-level impacts been acknowledged [@dudgeon_freshwater_2006; @murchie_fish_2008; @pringle_hydrologic_2001]. 

Changes to abiotic processes caused by river regulation can have a substantial impact on biotic communities. The negative effects of river regulation on migration and loss of spawning habitat (Fuller, Pope, Ashton, & Welsh, 2011; Sarah J. Kupferberg et al., 2012; Lind, Welsh, & Wilson, 1996; Rolls & Bond, 2017), reductions in population abundances and diversity (Fuller et al., 2011; Guzy, Eskew, Halstead, & Price, 2018; Lind et al., 1996; Sabo et al., 2017; Scribner et al., 2016; V√∂r√∂smarty et al., 2010; Zhong & Power, 1996), and fragmentation (Guzy et al., 2018; Sabo et al., 2017; Scribner et al., 2016; V√∂r√∂smarty et al., 2010; Werth, Sch√∂dl, & Scheidegger, 2014; Zhong & Power, 1996) have been well documented. However, most rivers have not been regulated for long periods (e.g., less than 100 years) compared to the time these organisms had to adapt to pre-anthropogenic river flow. In regulated rivers that organisms still occupy, it remains unknown whether populations can persist long-term with continued regulation. In other words, while some species may have persisted since regulation began in a system (e.g., several decades), this does not necessarily mean these populations will persist into the future under current flow regulation regimes. Thus, exploring the potential for long-term persistence of populations under different flow regimes is a crucial component for guiding conservation efforts yet remains a significant gap. 

One tool that can help address this gap is the integration of genetics and hydrology to better assess the impact of river regulation on aquatic organisms (Scribner et al., 2016). Although aquatic organisms are often difficult to count and monitor by conventional methods, genetic monitoring can be a powerful tool to assess population health by revealing factors such as fragmentation and population declines. It is widely recognized that reductions in population connectivity can increase isolation and inbreeding, leading to a potential "extinction vortex" (Gilpin & Soule, 1986), yet there is limited understanding of how flow alteration may impair the processes crucial for maintenance of genetic variation and thus adaptive capacity. In addition, there is a current pressing need for more effective and flexible watershed management tools, particularly in relation to monitoring aquatic populations and implementation of environmental flows (Grantham, Merenlender, & Resh, 2010). Thus, population genetics could be a powerful tool to understand the influence of different flow regimes on population health and this information could facilitate improved flow management to better protect aquatic populations.

The river-breeding foothill yellow-legged frog (Rana boylii; FYLF) historically occurred in lower and mid-elevation streams and rivers from Southern Oregon to northern Baja California west of the Sierra-Cascade crest (Stebbins, 2003). FYLF are intimately linked with river hydrology because they have evolved to spawn in synchrony with natural flow cues associated with seasonal spring snowmelt or rain recession periods (Bondi, Yarnell, Lind, & Lind, 2013; S. J. Kupferberg, 1996; S. M. Yarnell et al., 2010; S. Yarnell, Peek, Epke, & Lind, 2016). However, population declines have been documented across the former range of this species, particularly in southern California and the Sierra Nevada where it has been extirpated from approximately 50 percent of its historical range (Davidson, Shaffer, & Jennings, 2002; Jennings & Hayes, 1994). In California, particularly in the Sierra Nevada, river regulation may be a significant environmental stressor (Sarah J. Kupferberg et al., 2012; Lind et al., 1996). Regulated river reaches typically alter flows by augmenting or diverting winter and spring runoff, thereby reducing or eliminating flow cues and disrupting natural flow regimes. Aseasonal flow fluctuation from river regulation can scour (detach from substrate) or desiccate FYLF egg masses, and the loss of clutches may have a significant demographic impact because only one egg mass is laid per year. In many regulated rivers in the Sierra Nevada, FYLF populations are now restricted to small unregulated tributaries flowing into the regulated mainstem.

Here, we investigate the impacts of river regulation on genetic health of FYLF populations across three different flow regimes. Given that population connectivity and genetic diversity are known to be play critical roles in long-term species persistence, we explore the association between these metrics and levels of river regulation. Our goal is to assess the genetic health of FYLF under different river regulation regimes to better inform the potential for long-term persistence. Addressing this question will help to inform management and conservation efforts for FYLF, as well as the potential utility of genetics for future conservation monitoring efforts in aquatic species.

## Methods

### Sample collection

345 FYLF buccal or tissue samples were used in this study (see Table S1). Field sampling was conducted as previously described (Heyer, Donnelly, McDiarmid, Hayek, & Foster, 1994), under CDFW SCP Permit #0006881, with IACUC protocol #19327. Individual post-metamorphic frogs were buccal-swabbed following established protocols (Broquet, Berset-Braendli, Emaresi, & Fumagalli, 2007; Goldberg, Kaplan, & Schwalbe, 2003; Pidancier, Miquel, & Miaud, 2003). Each post-metamorphic individual was comprehensively swabbed underneath tongue and cheek for approximately one minute. Swabs were air dried for approximately five minutes and placed in 1.5 mL microcentrifuge tubes while in the field. Samples were stored in the laboratory at -80¬∞C until DNA extraction. Where possible, tail clips from tadpole larvae were collected, and tadpoles greater than 15 mm total length were targeted (Parris et al., 2010; Wilbur & Semlitsch, 1990). One small (<3mm) tail clip was taken per individual tadpole and dried on Whatman qualitative filter paper (grade 1) and stored at room temperature. 

### De novo assembly

To produce a high-quality genomic resource for a frog species with a large genome size, we first interrogated a large fraction of the genome using RAD sequencing (Baird et al., 2008; Miller et al., 2007). Paired-end sequence data were generated from 24 FYLF individuals (sampling details given in Table S2) across coastal and Sierra Nevada populations from California, USA. DNA was extracted with a magnetic bead‚Äìbased protocol (Ali et al., 2016) and quantified using Quant-iT PicoGreen dsDNA Reagent (Thermo Fisher Scientific) with an FLx800 Fluorescence Reader (BioTek Instruments). RAD libraries were constructed using the SbfI restriction enzyme and a new RAD protocol (Ali et al., 2016). De novo loci discovery and contig extension were carried as previously described (Miller et al., 2012) using the alignment program Novoalign and the genome assembler PRICE (Ruby, Bellare, & Derisi, 2013). This pipeline resulted in a set of 77,544 RAD contigs ranging from 300 to 800 bp (Table S3) which served as a de novo partial genome reference for all subsequent downstream analyses.

### Rapture sequencing

We then performed Rapture on all samples (Table S1) (2016) using 8,533 RAD capture baits (120 bp) were designed by Arbor Biosciences from the de novo alignment (Table S4). The final Rapture library was sequenced in 50% of an Illumina HiSeq 3000 lane. Rapture sequence data from each individual (Table S1) were aligned against the de novo partial genome reference using the BWA-MEM algorithm (Li, 2013; Li & Durbin, 2010) and saved to BAM format. SAMtools was used to sort, filter for proper pairs, remove PCR duplicates, and index binary alignment map (BAM), as well as merge sequences from multiple libraries (Li et al., 2009). BAM files from the same sample were merged before indexing using SAMtools.

### Principal component analysis

A probabilistic framework was used to discover SNPs for PCA as it does not require calling genotypes and is suitable for low-coverage sequencing data (Fumagalli et al., 2013; Korneliussen, Moltke, Albrechtsen, & Nielsen, 2013). All Rapture analyses were conducted using Analysis of Next Generation Sequencing Data (ANGSD) (Korneliussen, Albrechtsen, & Nielsen, 2014). ANGSD analyses were conducted following methods from Prince et al (2017), with a minimum mapping quality score (minMapQ) of 10, a minimum base quality score (minQ) of 20, and the genotype likelihood model (GL 1) (Li, 2011). To maximize data quality, samples with less than 100,000 aligned reads were excluded (Table S1, S2) using and only sites represented in at least 50% of the included samples (minInd) were used. Settings used in ANGSD for PCA to identify polymorphic sites included a SNP_pval of 1e-6, inferring major and minor alleles (doMajorMinor 1), estimating allele frequencies (doMaf 2) (Kim et al., 2011), retaining SNPs with a minor allele frequency of at least 0.05 (minMaf), genotype posterior probabilities were calculated with a uniform prior (doPost 2), and the doIBS 1 and doCov 1 options were used to generate PCA data. Principal components (PC) summarizing population structure were derived from classic eigenvalue decomposition and were visualized using the ggplot2 package in R (R Core Team, 2017).

### Genetic differentiation and diversity estimates

Mean scaled FST was used to quantify genetic differentiation between populations (Rousset, 1997; Wright, 1943). Genome-wide FST between population pairs was estimated by first calculating a site frequency spectrum (SFS) for each population (doSaf) (Nielsen, Korneliussen, Albrechtsen, Li, & Wang, 2012) with ANGSD. The two-dimensional SFS and global FST between each population pair were then estimated using realSFS (Korneliussen et al., 2014). FST was calculated between each pair of collection locations within a watershed, and the mean of all pairwise calculations within that watershed was calculated for each location. We calculated the river distances (distance along river network) between locations within watersheds using the riverdist package in R (Tyers, 2017), and used the mean pairwise river distance (km) to all other locations within the watershed. These values were plotted and a generalized linear model was fitted (FST ~ Mean River Distance) in R (R Core Team, 2017). To calculate Watterson‚Äôs Œ∏S (Watterson, 1975), and Tajima‚Äôs Œ∏ùúã (Tajima, 1983), we used SFS that were estimated as described above as priors (pest) to calculate each statistic for each site (doThetas), which were averaged to obtain a single value for each statistic (Korneliussen et al., 2013). 

### Boosted regression tree modeling of variance in FST

We used boosted regression tree (BRT) models with the R packages gbm (Ridgeway, 2015) and dismo (Hijmans, Phillips, Leathwick, & Elith, 2017) to assess the relative influence of river regulation as compared to other covariates. Boosted regression trees (BRT) are suitable frameworks for large and complex ecological datasets because they do not assume normality, nor linear relationships between predictor and response variables and they ignore non-informative predictor variables (Graham et al., 2008; Steel, Peek, Lusardi, & Yarnell, 2017). BRTs use iterative boosting algorithms to combine simple decision trees to improve model performance (De‚Äôath, 2007) and provide a robust alternative to many traditional statistical methods (Guisan et al., 2007; Phillips, Anderson, & Schapire, 2006). BRTs assess the relative impact of modeled variables by calculating the number of times a variable is selected for splitting a tree across all folds of the cross validation. Following Steel et al. 2017, estimates of relative influence for each predictor variable were used to evaluate the relative contribution a variable had in predicting the response. To evaluate the relative influence of covariates on FST, models were trained using river distance (km), elevation (m), upstream drainage area (km2), Strahler stream order, and number of samples per location. Stream segment data on elevation, length, slope, stream order, and drainage area were derived from NHD Plus attributes (U.S. Geological Survey, National Hydrography Dataset, Digital data, accessed, August 2017 at http://nhd.usgs.gov/data.html). In addition, ŒîŒ∏ (Œ∏ùúã ‚àí Œ∏S) was included to assess the effect of genomic variation on FST across regulation types. 

Model training and fitting were conducted following methods previously described in (Steel et al., 2017). To reduce overfitting, the learning rate (also known as the shrinking rate) was set to 0.001. Stochastic gradient boosting was utilized to reduce prediction error (De‚Äôath, 2007) and the fraction of training data sampled to build each tree was 0.75, within the range as recommended by (Brown et al., 2012). Tree complexity was set to three to allow for second and third order interaction effects. The minimum number of observations required in the final nodes of each tree was three. A ten-fold cross-validation technique allowed us to determine the number of trees at which prediction error was minimized using the cross-validation deviance. Model performance was evaluated using the minimum estimated cross-validation deviance which maximized the estimated deviance explained.

\clearpage

## Results

**_Rapture produces high quality genomic data for FYLF_**

To begin investigating the impact of river regulation on FYLF, we collected frog tissue and buccal samples from 30 locations in six rivers representing three different flow impairment levels associated with hydropower generation. The three flow regimes assessed were: 1) hydropeaking, where flows are pulsed on most days from late spring through fall to provide electricity during peak-use hours and for recreational whitewater rafting; 2) bypass, which diverts river flows from an upstream portion of the basin to the downstream power generation facilities; and 3) unregulated, a largely natural flow regime where no upstream controls exist to regulate flows (Figure 1). Flow data were obtained for each river reach using proximal USGS gaging stations (Table S5). We sampled a total of 345 FYLF from sites in three major watersheds (Yuba, Bear, and American) in the northern Sierra Nevada of California (Figure 1A; Table 1). The six study rivers share a similar Mediterranean climate, underlying geology, watershed aspect (west-slope), stream morphology (riffle-pool), and vegetative communities, but differ in the intensity of flow regulation (Steel et al., 2017). Although river regulation occurs in all three of the study watersheds, both the North Yuba and North Fork (NF) American are unregulated whereas the Middle Fork (MF) American is the only river that has a hydropeaking flow regime (Figure 1A). 

To generate genetic data from the samples, we performed RAD Capture (a.k.a. Rapture) (Ali et al., 2016) on the samples by generating SbfI RAD libraries, capturing a subset of the RAD loci using 8,533 baits (see Methods), and sequencing the resulting library on an Illumina HiSeq. We then aligned the sequencing reads from each sample to a de novo RAD assembly (see Methods). The mean number of filtered alignments across all 345 samples was 324,928. For downstream analysis, we selected individuals that had greater than 100,000 alignments (n=277), which provided sufficient data to investigate population genetic attributes at broad and fine geographic scales (see below). FYLF are cryptic, and often occur in low densities within the study area. Thus, we retained a minimum of three individuals per site, and the mean number of samples per site was approximately nine (Table 1). With genomic data, population genetic parameters can be accurately estimated from even low sample numbers (Hotaling et al., 2018), and genomic analyses in non-model organism often use fewer loci (Narum, Buerkle, Davey, Miller, & Hohenlohe, 2013). We conclude that the sequence data we obtained should be appropriate for population genetic analyses across our study area.

**_Anomalous genetic pattern in highly regulated reach of Middle Fork American watershed_**

To assess FYLF population structure across the collection locations, we used ANGSD (Korneliussen et al., 2014) to discover 44,406 SNPs and perform principal component analysis (PCA; see Methods), which provides a dimensionless comparison of all samples. The first two principal components revealed four main groups corresponding to the Yuba, Bear, North Fork (NF) American, and Middle Fork (MF) American samples (Figure 2A). Unlike the Yuba watershed where all rivers clustered as one group, the two rivers within the American watershed (the NF American and MF American) were separated by both PC1 and PC2. Although the NF American watershed clustered closely with the adjacent Bear watershed, the MF American showed a surprisingly high degree of genetic differentiation from other locations (Figure 2A). These data suggest that there is less genetic differentiation between the NF American and the Bear watersheds, than between the NF and MF American watersheds. We conclude that measurements of overall genetic differentiation in FYLF from our study area largely conform to watershed and geographic expectations, with the exception of the American watershed, which shows a surprisingly high degree of genetic differentiation between the North (unregulated) and Middle (hydropeaking) Forks.

To further investigate patterns of genetic variation within the American Watershed, we performed two PCAs, one on samples from the NF American, and the other on samples from the MF American. The PCA of the NF American showed minimal differentiation among locations, with different study sites blending together and weak patterns of population structure (Figure 2B). In contrast, PCA of the MF American showed strong differentiation between sites (Figures 2C, 2D). The MF American PCA completely resolved all sites, with the first component (PC1) strongly differentiating the samples in the hydropeaking reach from all other sites in the MF American. This pattern may be due to the differential river regulation between the two rivers; the NF American is unregulated and has weak PCA differentiation, whereas the MF American has a higher level of river regulation and all sites form distinct genetic clusters, indicative of reduced gene flow among sites within the MF American.
River regulation is the strongest predictor of genetic isolation with FYLF in the Northern Sierra

To assess how patterns of genetic differentiation are associated with river regulation across our entire study area, we estimated pairwise FST (Wright, 1943) between all collection locations within a river for all six rivers. We then plotted the scaled mean pairwise FST [mean FST / (1-mean FST)] (Rousset, 1997) for each location against the mean river distance (the average distance along the river network from each collection location to every other location within that study river). Furthermore, each location was categorized by regulation level of closest mainstem location (see Methods). While there was a clear relationship between FST and river distance (as shown by the slope of regression lines in Figure 3A), there was a striking pattern of elevated FST by regulation type (Figure 3A). Even the bypass regulation type showed a distinct pattern of elevated FST. For instance, regulated rivers with locations separated by less than 10km had FST values comparable to unregulated locations separated by mean river distances over 30 km. Hydropeaking was the most extreme pattern of the three regulation types and showed highly elevated FST values with the steepest regression coefficient. The baseline FST or global mean increased by over 0.1 between the unregulated (mean FST=0.141), and regulated locations (global mean for bypass FST=0.256, hydropeaking FST=0.278). This suggests a greater degree of isolation within sites in regulated river reaches compared with FYLF populations in unregulated reaches as larger FST values represent reductions in heterozygosity due to population subdivision (Slatkin, 1987). We conclude FYLF in regulated rivers show patterns of greater population isolation and loss of heterozygosity compared to frogs in unregulated locations.

To investigate the relative influence of river regulation compared to other covariates such as river distance on genetic differentiation (i.e. FST), we used boosted regression tree (BRT) modeling. Covariates included flow regime alteration type, river distance, watershed variables derived from National hydrology data (NHD), topographic data, and allele frequency spectrum skew (see below, Methods). We found flow regulation explained the greatest amount of variance in FST (Figure 3B). Thus, river regulation has a larger relative influence than mean river distance between sampling locations, which is often the most important factor influencing genetic differentiation (Rousset, 1997; Slatkin, 1987; Wright, 1943). We conclude there is a pattern of isolation and limited connectivity between populations in regulated reaches.

**_River regulation strongly correlated with decreasing genetic diversity in FYLF_**
To
 investigate the association between river regulation and genetic diversity trajectory (stable, increasing, or decreasing), we summarized patterns of genetic variation using two estimators of Œ∏ (4NŒº): Tajima‚Äôs Œ∏ (Œ∏ùúã) is based on the average number of pairwise differences (Tajima, 1983), and Watterson‚Äôs Œ∏ (Œ∏S) is based on the number of segregating sites (Watterson, 1975) (see methods). These estimators are influenced by the demographic history of a population and provide information on the trajectory of changes in genetic diversity. When genetic diversity has been stable, these estimates should be equal; when genetic diversity has been increasing, Œ∏ùúã > Œ∏S; and when genetic diversity has been decreasing, Œ∏S > Œ∏ùúã. We found zero populations sampled within regulated watersheds had evidence of increasing genetic diversity (e.g., a Œ∏ùúã ‚àí Œ∏S that was less than zero) (Figure 4A). The regulated locations showed a clear trajectory of genetic diversity loss (Figure 4A, 4B). Three of the four hydropeaking locations had the highest values of ŒîŒ∏ (Œ∏ùúã ‚àí Œ∏S), and the global mean was significantly different from other regulation types. Although some tributary populations within unregulated watersheds also showed signs of genetic diversity loss, the mean genetic diversity trajectory at unregulated locations was largely neutral (Figure 4B). This indicates populations in the northern Sierra Nevada which are already limited in number are losing genetic variation, and river regulation appears to be exacerbating these patterns. We conclude there is evidence of recent genetic diversity loss across populations in the regulated river systems, regardless of regulation type.

## Discussion

Although massive parallel sequencing (MPS) technologies have the potential to facilitate collection of high-quality genetic data in virtually any species, a number of challenges still remain for many species including low quality or non-existent reference genomes, large/complex/repetitive genomes, and high cost of processing/sequencing in studies with many samples. Amphibians are particularly challenging as many species have very large genome sizes (Nunziata, Lance, Scott, Lemmon, & Weisrock, 2017) for example, there are only two frog reference genome assemblies available as of 2018 (Hellsten et al., 2010; Sun et al., 2015). Our results demonstrate that Rapture (Ali et al., 2016) is a suitable method to rapidly and inexpensively discover a large number of loci in a frog species with a complex genome. In this study, we used new RAD sequencing and RAD capture (Rapture) methods (Ali et al., 2016) to generate high-quality genomic data suitable for discovering and genotyping many single nucleotide polymorphisms (SNPs) in FYLF. Based on this dataset, we were able to successfully characterize patterns of genetic variation within FYLF as well as design a set of RAD capture baits that can be used as a genetic monitoring resource for FYLF (and likely other ranid species). This highlights that the collection of genetic information, even from large numbers of samples or in complex genomes, is no longer a limitation with current genomic methods such as RAD and Rapture.

Demographic connectivity is widely recognized as a fundamental driver of long-term population persistence (Fahrig & Merriam, 1985; Taylor, Fahrig, Henein, & Merriam, 1993). Populations must adapt over time and connectivity is a major way to transfer genetic information. For example, previous studies have shown that adaptation can occur by spreading specific alleles across large geographic distances (Miller et al., 2012; Prince et al., 2017). In many regulated river reaches in the Sierra Nevada, FYLF now occur in isolated locations, breeding in tributaries rather than mainstem habitats. However, since these frogs have the potential to move long distances (FYLF have been observed moving over 1 km per day (Bourque, 2008)), the extent to which current population connectivity has been lost due to river regulation remains unknown. Examining pairwise FST, revealed a major decrease in connectivity in populations in regulated systems, even with limited river regulation (i.e., bypass reaches). Usually isolation by distance patterns best describe variation in genetic data, yet the primary factor influencing genetic differentiation among these rivers is hydrologic alteration (Figure 3B). Thus, despite being able to move long distances, FYLF have not been able to maintain population connectivity in regulated rivers. This demonstrates that even in species that can move relatively long distances and pass potential physical barriers (e.g., infrastructure such as dams, canals, and reservoirs likely do not represent barriers to movement of FYLF) loss of connectivity may still occur and can be revealed with genetic analysis. 

Genetic diversity is also a critical component for long-term population persistence because it is closely related to the evolutionary capacity for adaptation to environmental changes (Frankham, 2002; Hoffmann & Sgr√≤, 2011; Ishiyama, Koizumi, Yuta, & Nakamura, 2015; Lande & Shannon, 1996). In some cases, isolated populations can maintain genetic diversity when they are sufficiently sized (Whiteley et al., 2010), however, species of conservation concern typically have small and/or declining populations and thus may be susceptible to genetic diversity loss (Frankham, 2002; Krohn et al., 2018). Throughout the Sierra Nevada, FYLF have largely disappeared from regulated mainstem rivers, but the extent to which existing populations have been able to maintain genetic diversity is unclear. Strikingly, our analysis revealed that every single population within the regulated watersheds exhibits a trajectory of genetic diversity loss. Thus, genomic analysis of molecular variation provides a powerful lens to discover and assess trajectories of genetic diversity.

Our analyses, using metrics that serve as a reasonable proxy for genetic health, does not bode well for the long-term persistence of FYLF populations in regulated rivers in the Sierra Nevada. Many of these FYLF populations are already losing genetic diversity and given their small size and reduced connectivity the effects of inbreeding will likely exacerbate their problems. FYLF have evolved in river systems with consistent hydrologic seasonality and predictability, despite inter-annual variation. Flow regulation has altered patterns of hydrologic seasonality and predictability in many watersheds (Sarah J. Kupferberg et al., 2012). Long-term population persistence may still be possible if conservation efforts utilize methods that promote or maintain genetic health and increase population connectivity. For example, simulations by Botero et al. (2015) demonstrated adaptation persisted in modeled populations through large environmental changes‚Äîif phenotypic strategies were appropriate before and after the change‚Äîbut modeled populations declined rapidly when the current strategy was a mismatch to the current environment. Thus, FYLF conservation efforts should focus on river reaches where flow management may provide opportunities to more closely mimic local natural flow regimes and thus improve hydrologic connectivity.

Detecting evolutionary responses to within- and among-year changes in an ecological or hydrological context has previously been difficult. However, utilizing genetic data can fill these gaps and provide a highly informative process for identifying the impacts of anthropogenic and environmental change on the process of adaptation (Botero et al., 2015; Kahilainen, Puurtinen, & Kotiaho, 2014). We demonstrate that an aquatic species that has adapted to local hydrology patterns shows poor genetic health (i.e., clear patterns of decreased connectivity and trajectories of genetic diversity loss). Our results highlight the potential impact of river regulation on aquatic organisms and their potential for long term persistence. In the future, similar genetic approaches could be used in many other contexts to explore the impacts of river regulation on aquatic organisms. Taken together, our results demonstrate that genetic monitoring can be a powerful tool for assessment of population health and should be a critical component of conservation management in aquatic organisms.

**Data Archiving Statement:** Should the manuscript be accepted, the data supporting the results will be archived in an appropriate public repository such as Dryad, and the data DOI will be appended to the end of the article. 

```{r maxdelays, results="asis", echo=Fevalrary(knitr)
kable(max_delays, 
      col.names = c("Airline", "Max Arrival Delay"),
      caption = "Maximum Delays by Airline",
      caption.short = "Max Delays by Airline",
      longtable = TRUE,
      booktabs = TRUE)
```

The last two options make the table a little easier-to-read.



