%%%---PREAMBLE---%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\documentclass[twoside,12pt,final]{ucthesis-CA2012}
% fix for pandoc 1.14
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

%--- Packages ---------------------------------------------------------
\usepackage[lofdepth,lotdepth,caption=false]{subfig}
\usepackage{fancyhdr}
\usepackage{amsmath, amssymb, graphicx}
\usepackage{xspace}
\usepackage{braket}
\usepackage{color}
\usepackage{setspace}
\usepackage{fancyvrb}
\usepackage{array}
\usepackage{ifxetex,ifluatex}
\usepackage{etoolbox}
\usepackage{booktabs}
\usepackage{longtable}

% suppress bottom page numbers on first page of each chapter
% because they overlap with text
%\patchcmd{\chapter}{plain}{empty}{}{}

%---New Definitions and Commands------------------------------------------------------

\newtheorem{theorem}{Jibberish}

\bibliography{references}

\hyphenation{mar-gin-al-ia}

% from uw_template.tex

% commands and environments needed by pandoc snippets
% extracted from the output of `pandoc -s`
%% Make R markdown code chunks work

\ifxetex
  \usepackage{fontspec,xltxtra,xunicode}
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
\else
  \ifluatex
    \usepackage{fontspec}
    \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \else
    \usepackage[utf8]{inputenc}
  \fi
\fi
\DefineShortVerb[commandchars=\\\{\}]{\|}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\newenvironment{Shaded}{}{}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.56,0.13,0.00}{{#1}}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.25,0.63,0.44}{{#1}}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.25,0.63,0.44}{{#1}}}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.25,0.63,0.44}{{#1}}}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.25,0.44,0.63}{{#1}}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.25,0.44,0.63}{{#1}}}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.38,0.63,0.69}{\textit{{#1}}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{{#1}}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{1.00,0.00,0.00}{\textbf{{#1}}}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.02,0.16,0.49}{{#1}}}
\newcommand{\RegionMarkerTok}[1]{{#1}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{1.00,0.00,0.00}{\textbf{{#1}}}}
\newcommand{\NormalTok}[1]{{#1}}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}
\newcommand{\BuiltInTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}

\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex,
              colorlinks=true,
              linkcolor=blue]{hyperref}
\else
  \usepackage[unicode=true,
              colorlinks=true,
              linkcolor=blue]{hyperref}
\fi
\hypersetup{breaklinks=true, pdfborder={0 0 0}}
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{0}

%---Set Margins ------------------------------------------------------
\setlength\oddsidemargin{0.25 in} \setlength\evensidemargin{0.25 in} \setlength\textwidth{6.25 in} \setlength\textheight{8.5 in} %8.5
\setlength\footskip{0.25 in} \setlength\topmargin{0 in} \setlength\headheight{0.25 in} \setlength\headsep{0.25 in}

%%%---DOCUMENT---%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

%=== Preliminary Pages ============================================
\begin{ucfrontmatter}

  %%%%%%%%%%%%%%%%%%%%%%%%%%%
  % TITLE PAGE INFORMATION %
  %%%%%%%%%%%%%%%%%%%%%%%%%%%

  \title{Linking watersheds with genomics: Population genetics of the Foothill
yellow-legged Frog (\emph{Rana boylii})}

  \author{Ryan A Peek}
  \prevdegreeA{B.S. (University of California, Davis) 2002}
  \prevdegreeB{M.S. (University of San Francisco) 2010}
  \report{DISSERTATION} 
  \degree{DOCTOR OF PHILOSOPHY} 
  \degreemonth{September} \degreeyear{2018}
  %\defensemonth{September} % should be one of the following: March,
  %\defenseyear{2018}
  \chair{Michael R. Miller}  % this is your advisor
  \othermemberA{Peter B. Moyle} % This is a member of your committee
  \othermemberB{Mark W. Schwartz} % This is a member of your committee
  \othermemberC{} % This is a member of your committee (if your department requires 4 members)
  \numberofmembers{3} % should match the number of entries above (chair + othermembers)

  \field{Ecology}
  \campus{Davis}
	\maketitle
	\approvalpage
	%\copyrightpage

  %%%%%%%%%%%%%%%%%%%%%%%%%%%
  % DEDICATION PAGE INFORMATION %
  %%%%%%%%%%%%%%%%%%%%%%%%%%%
    \begin{dedication}

      \vspace*{25ex}
      \begin{center}
      \begin{Large}

        To those who love the water

      \end{Large}
      \end{center}
  \end{dedication}
  \begin{acknowledgements}
    Thanks everyone!
  \end{acknowledgements}
	%%%%%%%%%%%%%%%%%%%%%%%%%%%
  % ABSTRACT %
  %%%%%%%%%%%%%%%%%%%%%%%%%%%
  \begin{abstract}
    \addcontentsline{toc}{chapter}{Abstract}
    %todo: max 350 words

    \emph{Rana boylii} is an imperiled frog species native to CA and OR, and
    it is currently designated as a species of special concern (CDFW) in the
    state of CA. It has been petitioned as candidate for federal (USFWS) and
    state (CDFW) listing. As a lotic breeding amphibian, \emph{R. boylii} is
    tied closely to the local hydrology within the watersheds it inhabits.
    It is particularly sensitive to alterations to flow regimes. Future
    conservation of this species requires several crucial components,
    including establishing higher resolution watershed-level boundaries for
    populations across the species range, delineation of distinct population
    segments that can be utilized in conservation management, and
    quantification of relative genomic health of these groups (i.e., genomic
    diversity, measures of isolation, connectivity). We provide data on over
    700 samples across the range of the species, identify six genomically
    distinct groups, as well as subdivision at local watershed scales. These
    data are congruent with other molecular data on \emph{R. boylii},
    although we identify and add individuals from the Feather watershed as a
    distinctive group, at both population and subpopulation scales.
    Quantification and comparison of genomic variation across populations
    indicates populations in the southern coast, southern Sierra Nevada, and
    Northern Sierra/Feather basin in California should have high
    prioritization in conservation efforts due to patterns of bottlenecking
    and low genomic diversity. One major impact on \emph{R. boylii}
    populations has been river regulation, or the hydrological alteration of
    flow by dams and diversions. River regulation has been implicated as a
    cause of fundamental changes to downstream aquatic ecosystems.
    Regulation changes the natural flow regime which may restrict population
    connectivity and decrease genetic diversity in some species. Since
    population connectivity and the maintenance of genetic diversity are
    fundamental drivers of long-term persistence, understanding the extent
    that river regulation impacts these critical attributes of genetic
    health is an important goal. Foothill yellow-legged frog (FYLF;
    \emph{Rana boylii}) were historically abundant throughout many western
    rivers but have declined since the onset of regulation. However, the
    extent to which FYLF populations in regulated rivers are maintaining
    connectivity and genetic diversity is unknown. Here we use genetic
    methods to investigate the impacts of river regulation on FYLF to
    explore their potential for long-term persistence under continued
    regulation. We found FYLF in regulated rivers showed striking patterns
    of isolation and trajectories of genetic diversity loss relative to
    unregulated rivers. For example, river regulation explained the greatest
    amount of variance in population genetic differentiation compared with
    other covariates including geographic distance. Importantly, patterns of
    connectivity and genetic diversity loss were observed regardless of
    regulation level but were most prominent in locations with the greatest
    regulation intensity. Although our results do not bode well for
    long-term persistence of FYLF populations under current flow regulation
    regimes, they do highlight the power of genetic monitoring for assessing
    population health in aquatic organisms. In addition, analyses of data on
    hybridization between \emph{R. boylii} and \emph{R. sierrae} in a single
    region of the Sierra Nevada of California is shown. Cross-breeding
    between species, or hybridization in vertebrates has generally been
    considered a rare occurrence. Hybridization between species may combine
    parental genotypes in ways that yield reproductively sterile or isolated
    lineages, and hybridization events may be short-lived and difficult to
    detect. Furthermore, cross-breeding or hybridization between closely
    related taxa can promote gene flow (introgression) between species,
    which may be an important evolutionary mechanism for either
    homogenization (reversing initial divergence between species) or
    reproductive isolation (potentially leading to speciation). Here, we
    used thousands of genetic markers from nuclear DNA to detect a very rare
    hybridization event between two co-occurring frog species (\emph{R.
    boylii} and \emph{R. sierrae}) in the Sierra Nevada of California.
    Hybridization events are rare but have occurred in the Feather drainage
    between these two species, however it appears these are terminal events
    based on PCA, admixture, and tests of heterozygosity in species
    diagnostic SNPs.

    %\abstractsignature
  \end{abstract}
	\tableofcontents
\end{ucfrontmatter}
\begin{ucmainmatter}

\hypertarget{introduction}{%
\chapter*{Introduction}\label{introduction}}
\addcontentsline{toc}{chapter}{Introduction}

Welcome to the \emph{R Markdown} thesis template. This template is based
on (and in many places copied directly from) the UW LaTeX template, but
hopefully it will provide a nicer interface for those that have never
used TeX or LaTeX before. Using \emph{R Markdown} will also allow you to
easily keep track of your analyses in \textbf{R} chunks of code, with
the resulting plots and output included as well. The hope is this
\emph{R Markdown} template gets you in the habit of doing reproducible
research, which benefits you long-term as a researcher, but also will
greatly help anyone that is trying to reproduce or build onto your
results down the road.

Hopefully, you won't have much of a learning period to go through and
you will reap the benefits of a nicely formatted thesis. The use of
LaTeX in combination with \emph{Markdown} is more consistent than the
output of a word processor, much less prone to corruption or crashing,
and the resulting file is smaller than a Word file. While you may have
never had problems using Word in the past, your thesis is likely going
to be at least twice as large and complex as anything you've written
before, taxing Word's capabilities. After working with \emph{Markdown}
and \textbf{R} together for a few weeks, we are confident this will be
your reporting style of choice going forward.

\textbf{Why use it?}

\emph{R Markdown} creates a simple and straightforward way to interface
with the beauty of LaTeX. Packages have been written in \textbf{R} to
work directly with LaTeX to produce nicely formatting tables and
paragraphs. In addition to creating a user friendly interface to LaTeX,
\emph{R Markdown} also allows you to read in your data, to analyze it
and to visualize it using \textbf{R} functions, and also to provide the
documentation and commentary on the results of your project. Further, it
allows for \textbf{R} results to be passed inline to the commentary of
your results. You'll see more on this later.

\textbf{Who should use it?}

Anyone who needs to use data analysis, math, tables, a lot of figures,
complex cross-references, or who just cares about the final appearance
of their document should use \emph{R Markdown}. Of particular use should
be anyone in the sciences, but the user-friendly nature of
\emph{Markdown} and its ability to keep track of and easily include
figures, automatically generate a table of contents, index, references,
table of figures, etc. should make it of great benefit to nearly anyone
writing a thesis project.

\hypertarget{reg-health}{%
\chapter{Flow regulation associated with decreased genetic health of a
river-breeding frog species}\label{reg-health}}

\hypertarget{introduction-1}{%
\section{Introduction}\label{introduction-1}}

Rivers simultaneously connect and carve the landscapes through which
they flow. Rivers provide corridors of connectivity for riparian and
aquatic organisms such as fish, amphibians, and macroinvertebrates
(Wiens 2002, Pringle 2003), while also acting as physical barriers on
the landscape for many terrestrial organisms (Voelker et al. 2013, Caz√©
et al. 2016). Hydrologic connectivity (Pringle 2003) transfers energy,
organisms and ultimately genetic variation and thus is a critical
component for population persistence in dynamic systems where
populations must constantly adapt to temporal and spatial changes. In
Mediterranean climates, rivers have strong seasonal patterns associated
with cold, wet winters and warm, dry summers. Native aquatic organisms
have evolved life histories well adapted to these natural patterns,
which are both predictable and seasonal (Yarnell et al. 2010, Tonkin et
al. 2017).

River regulation, or the hydrological alteration of flow by dams and
diversions, impacts the seasonal and interannual flow variability within
a watershed. Regulation changes the natural flow regime and dramatically
alters geomorphic and hydrologic connectivity of watersheds (Poff et al.
2007), which may restrict natural population connectivity (Schick and
Lindley 2007, Shaw et al. 2016). River regulation can change flow
frequency, magnitude, duration, timing, and rate of change, which can
have significant impacts on aquatic organisms and ecological processes
(Poff et al. 2007, Yarnell et al. 2010). River regulation, and more
specifically, regulation associated with hydropower generation, has been
implicated as a cause of fundamental changes to downstream aquatic
ecosystems (Power et al. 1996, Bunn and Arthington 2002, Moyle et al.
2011). The hydrological regimes of over half of the world's largest
rivers have been altered by large dams (Nilsson et al. 2005) and only
recently has the extent of flow alteration and the associated
ecosystem-level impacts been acknowledged (Pringle 2001, Dudgeon et al.
2006, Murchie et al. 2008).

Changes to abiotic processes caused by river regulation can have a
substantial impact on biotic communities. The negative effects of river
regulation on migration and loss of spawning habitat (Fuller, Pope,
Ashton, \& Welsh, 2011; Sarah J. Kupferberg et al., 2012; Lind, Welsh,
\& Wilson, 1996; Rolls \& Bond, 2017), reductions in population
abundances and diversity (Fuller et al., 2011; Guzy, Eskew, Halstead, \&
Price, 2018; Lind et al., 1996; Sabo et al., 2017; Scribner et al.,
2016; V√∂r√∂smarty et al., 2010; Zhong \& Power, 1996), and fragmentation
(Guzy et al., 2018; Sabo et al., 2017; Scribner et al., 2016; V√∂r√∂smarty
et al., 2010; Werth, Sch√∂dl, \& Scheidegger, 2014; Zhong \& Power, 1996)
have been well documented. However, most rivers have not been regulated
for long periods (e.g., less than 100 years) compared to the time these
organisms had to adapt to pre-anthropogenic river flow. In regulated
rivers that organisms still occupy, it remains unknown whether
populations can persist long-term with continued regulation. In other
words, while some species may have persisted since regulation began in a
system (e.g., several decades), this does not necessarily mean these
populations will persist into the future under current flow regulation
regimes. Thus, exploring the potential for long-term persistence of
populations under different flow regimes is a crucial component for
guiding conservation efforts yet remains a significant gap.

One tool that can help address this gap is the integration of genetics
and hydrology to better assess the impact of river regulation on aquatic
organisms (Scribner et al., 2016). Although aquatic organisms are often
difficult to count and monitor by conventional methods, genetic
monitoring can be a powerful tool to assess population health by
revealing factors such as fragmentation and population declines. It is
widely recognized that reductions in population connectivity can
increase isolation and inbreeding, leading to a potential ``extinction
vortex'' (Gilpin \& Soule, 1986), yet there is limited understanding of
how flow alteration may impair the processes crucial for maintenance of
genetic variation and thus adaptive capacity. In addition, there is a
current pressing need for more effective and flexible watershed
management tools, particularly in relation to monitoring aquatic
populations and implementation of environmental flows (Grantham,
Merenlender, \& Resh, 2010). Thus, population genetics could be a
powerful tool to understand the influence of different flow regimes on
population health and this information could facilitate improved flow
management to better protect aquatic populations.

The river-breeding foothill yellow-legged frog (Rana boylii; FYLF)
historically occurred in lower and mid-elevation streams and rivers from
Southern Oregon to northern Baja California west of the Sierra-Cascade
crest (Stebbins, 2003). FYLF are intimately linked with river hydrology
because they have evolved to spawn in synchrony with natural flow cues
associated with seasonal spring snowmelt or rain recession periods
(Bondi, Yarnell, Lind, \& Lind, 2013; S. J. Kupferberg, 1996; S. M.
Yarnell et al., 2010; S. Yarnell, Peek, Epke, \& Lind, 2016). However,
population declines have been documented across the former range of this
species, particularly in southern California and the Sierra Nevada where
it has been extirpated from approximately 50 percent of its historical
range (Davidson, Shaffer, \& Jennings, 2002; Jennings \& Hayes, 1994).
In California, particularly in the Sierra Nevada, river regulation may
be a significant environmental stressor (Sarah J. Kupferberg et al.,
2012; Lind et al., 1996). Regulated river reaches typically alter flows
by augmenting or diverting winter and spring runoff, thereby reducing or
eliminating flow cues and disrupting natural flow regimes. Aseasonal
flow fluctuation from river regulation can scour (detach from substrate)
or desiccate FYLF egg masses, and the loss of clutches may have a
significant demographic impact because only one egg mass is laid per
year. In many regulated rivers in the Sierra Nevada, FYLF populations
are now restricted to small unregulated tributaries flowing into the
regulated mainstem.

Here, we investigate the impacts of river regulation on genetic health
of FYLF populations across three different flow regimes. Given that
population connectivity and genetic diversity are known to be play
critical roles in long-term species persistence, we explore the
association between these metrics and levels of river regulation. Our
goal is to assess the genetic health of FYLF under different river
regulation regimes to better inform the potential for long-term
persistence. Addressing this question will help to inform management and
conservation efforts for FYLF, as well as the potential utility of
genetics for future conservation monitoring efforts in aquatic species.

\hypertarget{methods}{%
\section{Methods}\label{methods}}

\hypertarget{sample-collection}{%
\subsection{Sample collection}\label{sample-collection}}

345 FYLF buccal or tissue samples were used in this study (see Table
S1). Field sampling was conducted as previously described (Heyer,
Donnelly, McDiarmid, Hayek, \& Foster, 1994), under CDFW SCP Permit
\#0006881, with IACUC protocol \#19327. Individual post-metamorphic
frogs were buccal-swabbed following established protocols (Broquet,
Berset-Braendli, Emaresi, \& Fumagalli, 2007; Goldberg, Kaplan, \&
Schwalbe, 2003; Pidancier, Miquel, \& Miaud, 2003). Each
post-metamorphic individual was comprehensively swabbed underneath
tongue and cheek for approximately one minute. Swabs were air dried for
approximately five minutes and placed in 1.5 mL microcentrifuge tubes
while in the field. Samples were stored in the laboratory at -80¬∞C until
DNA extraction. Where possible, tail clips from tadpole larvae were
collected, and tadpoles greater than 15 mm total length were targeted
(Parris et al., 2010; Wilbur \& Semlitsch, 1990). One small
(\textless{}3mm) tail clip was taken per individual tadpole and dried on
Whatman qualitative filter paper (grade 1) and stored at room
temperature.

\hypertarget{de-novo-assembly}{%
\subsection{De novo assembly}\label{de-novo-assembly}}

To produce a high-quality genomic resource for a frog species with a
large genome size, we first interrogated a large fraction of the genome
using RAD sequencing (Baird et al., 2008; Miller et al., 2007).
Paired-end sequence data were generated from 24 FYLF individuals
(sampling details given in Table S2) across coastal and Sierra Nevada
populations from California, USA. DNA was extracted with a magnetic
bead--based protocol (Ali et al., 2016) and quantified using Quant-iT
PicoGreen dsDNA Reagent (Thermo Fisher Scientific) with an FLx800
Fluorescence Reader (BioTek Instruments). RAD libraries were constructed
using the SbfI restriction enzyme and a new RAD protocol (Ali et al.,
2016). De novo loci discovery and contig extension were carried as
previously described (Miller et al., 2012) using the alignment program
Novoalign and the genome assembler PRICE (Ruby, Bellare, \& Derisi,
2013). This pipeline resulted in a set of 77,544 RAD contigs ranging
from 300 to 800 bp (Table S3) which served as a de novo partial genome
reference for all subsequent downstream analyses.

\hypertarget{rapture-sequencing}{%
\subsection{Rapture sequencing}\label{rapture-sequencing}}

We then performed Rapture on all samples (Table S1) (2016) using 8,533
RAD capture baits (120 bp) were designed by Arbor Biosciences from the
de novo alignment (Table S4). The final Rapture library was sequenced in
50\% of an Illumina HiSeq 3000 lane. Rapture sequence data from each
individual (Table S1) were aligned against the de novo partial genome
reference using the BWA-MEM algorithm (Li, 2013; Li \& Durbin, 2010) and
saved to BAM format. SAMtools was used to sort, filter for proper pairs,
remove PCR duplicates, and index binary alignment map (BAM), as well as
merge sequences from multiple libraries (Li et al., 2009). BAM files
from the same sample were merged before indexing using SAMtools.

\hypertarget{principal-component-analysis}{%
\subsection{Principal component
analysis}\label{principal-component-analysis}}

A probabilistic framework was used to discover SNPs for PCA as it does
not require calling genotypes and is suitable for low-coverage
sequencing data (Fumagalli et al., 2013; Korneliussen, Moltke,
Albrechtsen, \& Nielsen, 2013). All Rapture analyses were conducted
using Analysis of Next Generation Sequencing Data (ANGSD) (Korneliussen,
Albrechtsen, \& Nielsen, 2014). ANGSD analyses were conducted following
methods from Prince et al (2017), with a minimum mapping quality score
(minMapQ) of 10, a minimum base quality score (minQ) of 20, and the
genotype likelihood model (GL 1) (Li, 2011). To maximize data quality,
samples with less than 100,000 aligned reads were excluded (Table S1,
S2) using and only sites represented in at least 50\% of the included
samples (minInd) were used. Settings used in ANGSD for PCA to identify
polymorphic sites included a SNP\_pval of 1e-6, inferring major and
minor alleles (doMajorMinor 1), estimating allele frequencies (doMaf 2)
(Kim et al., 2011), retaining SNPs with a minor allele frequency of at
least 0.05 (minMaf), genotype posterior probabilities were calculated
with a uniform prior (doPost 2), and the doIBS 1 and doCov 1 options
were used to generate PCA data. Principal components (PC) summarizing
population structure were derived from classic eigenvalue decomposition
and were visualized using the ggplot2 package in R (R Core Team, 2017).

\hypertarget{genetic-differentiation-and-diversity-estimates}{%
\subsection{Genetic differentiation and diversity
estimates}\label{genetic-differentiation-and-diversity-estimates}}

Mean scaled FST was used to quantify genetic differentiation between
populations (Rousset, 1997; Wright, 1943). Genome-wide FST between
population pairs was estimated by first calculating a site frequency
spectrum (SFS) for each population (doSaf) (Nielsen, Korneliussen,
Albrechtsen, Li, \& Wang, 2012) with ANGSD. The two-dimensional SFS and
global FST between each population pair were then estimated using
realSFS (Korneliussen et al., 2014). FST was calculated between each
pair of collection locations within a watershed, and the mean of all
pairwise calculations within that watershed was calculated for each
location. We calculated the river distances (distance along river
network) between locations within watersheds using the riverdist package
in R (Tyers, 2017), and used the mean pairwise river distance (km) to
all other locations within the watershed. These values were plotted and
a generalized linear model was fitted (FST \textasciitilde{} Mean River
Distance) in R (R Core Team, 2017). To calculate Watterson's Œ∏S
(Watterson, 1975), and Tajima's Œ∏ùúã (Tajima, 1983), we used SFS that were
estimated as described above as priors (pest) to calculate each
statistic for each site (doThetas), which were averaged to obtain a
single value for each statistic (Korneliussen et al., 2013).

\hypertarget{boosted-regression-tree-modeling-of-variance-in-fst}{%
\subsection{Boosted regression tree modeling of variance in
FST}\label{boosted-regression-tree-modeling-of-variance-in-fst}}

We used boosted regression tree (BRT) models with the R packages gbm
(Ridgeway, 2015) and dismo (Hijmans, Phillips, Leathwick, \& Elith,
2017) to assess the relative influence of river regulation as compared
to other covariates. Boosted regression trees (BRT) are suitable
frameworks for large and complex ecological datasets because they do not
assume normality, nor linear relationships between predictor and
response variables and they ignore non-informative predictor variables
(Graham et al., 2008; Steel, Peek, Lusardi, \& Yarnell, 2017). BRTs use
iterative boosting algorithms to combine simple decision trees to
improve model performance (De'ath, 2007) and provide a robust
alternative to many traditional statistical methods (Guisan et al.,
2007; Phillips, Anderson, \& Schapire, 2006). BRTs assess the relative
impact of modeled variables by calculating the number of times a
variable is selected for splitting a tree across all folds of the cross
validation. Following Steel et al.~2017, estimates of relative influence
for each predictor variable were used to evaluate the relative
contribution a variable had in predicting the response. To evaluate the
relative influence of covariates on FST, models were trained using river
distance (km), elevation (m), upstream drainage area (km2), Strahler
stream order, and number of samples per location. Stream segment data on
elevation, length, slope, stream order, and drainage area were derived
from NHD Plus attributes (U.S. Geological Survey, National Hydrography
Dataset, Digital data, accessed, August 2017 at
\url{http://nhd.usgs.gov/data.html}). In addition, ŒîŒ∏ (Œ∏ùúã ‚àí Œ∏S) was
included to assess the effect of genomic variation on FST across
regulation types.

Model training and fitting were conducted following methods previously
described in (Steel et al., 2017). To reduce overfitting, the learning
rate (also known as the shrinking rate) was set to 0.001. Stochastic
gradient boosting was utilized to reduce prediction error (De'ath, 2007)
and the fraction of training data sampled to build each tree was 0.75,
within the range as recommended by (Brown et al., 2012). Tree complexity
was set to three to allow for second and third order interaction
effects. The minimum number of observations required in the final nodes
of each tree was three. A ten-fold cross-validation technique allowed us
to determine the number of trees at which prediction error was minimized
using the cross-validation deviance. Model performance was evaluated
using the minimum estimated cross-validation deviance which maximized
the estimated deviance explained.

\clearpage

\hypertarget{results}{%
\section{Results}\label{results}}

\textbf{\emph{Rapture produces high quality genomic data for FYLF}}

To begin investigating the impact of river regulation on FYLF, we
collected frog tissue and buccal samples from 30 locations in six rivers
representing three different flow impairment levels associated with
hydropower generation. The three flow regimes assessed were: 1)
hydropeaking, where flows are pulsed on most days from late spring
through fall to provide electricity during peak-use hours and for
recreational whitewater rafting; 2) bypass, which diverts river flows
from an upstream portion of the basin to the downstream power generation
facilities; and 3) unregulated, a largely natural flow regime where no
upstream controls exist to regulate flows (Figure 1). Flow data were
obtained for each river reach using proximal USGS gaging stations (Table
S5). We sampled a total of 345 FYLF from sites in three major watersheds
(Yuba, Bear, and American) in the northern Sierra Nevada of California
(Figure 1A; Table 1). The six study rivers share a similar Mediterranean
climate, underlying geology, watershed aspect (west-slope), stream
morphology (riffle-pool), and vegetative communities, but differ in the
intensity of flow regulation (Steel et al., 2017). Although river
regulation occurs in all three of the study watersheds, both the North
Yuba and North Fork (NF) American are unregulated whereas the Middle
Fork (MF) American is the only river that has a hydropeaking flow regime
(Figure 1A).

To generate genetic data from the samples, we performed RAD Capture
(a.k.a. Rapture) (Ali et al., 2016) on the samples by generating SbfI
RAD libraries, capturing a subset of the RAD loci using 8,533 baits (see
Methods), and sequencing the resulting library on an Illumina HiSeq. We
then aligned the sequencing reads from each sample to a de novo RAD
assembly (see Methods). The mean number of filtered alignments across
all 345 samples was 324,928. For downstream analysis, we selected
individuals that had greater than 100,000 alignments (n=277), which
provided sufficient data to investigate population genetic attributes at
broad and fine geographic scales (see below). FYLF are cryptic, and
often occur in low densities within the study area. Thus, we retained a
minimum of three individuals per site, and the mean number of samples
per site was approximately nine (Table 1). With genomic data, population
genetic parameters can be accurately estimated from even low sample
numbers (Hotaling et al., 2018), and genomic analyses in non-model
organism often use fewer loci (Narum, Buerkle, Davey, Miller, \&
Hohenlohe, 2013). We conclude that the sequence data we obtained should
be appropriate for population genetic analyses across our study area.

\textbf{\emph{Anomalous genetic pattern in highly regulated reach of
Middle Fork American watershed}}

To assess FYLF population structure across the collection locations, we
used ANGSD (Korneliussen et al., 2014) to discover 44,406 SNPs and
perform principal component analysis (PCA; see Methods), which provides
a dimensionless comparison of all samples. The first two principal
components revealed four main groups corresponding to the Yuba, Bear,
North Fork (NF) American, and Middle Fork (MF) American samples (Figure
2A). Unlike the Yuba watershed where all rivers clustered as one group,
the two rivers within the American watershed (the NF American and MF
American) were separated by both PC1 and PC2. Although the NF American
watershed clustered closely with the adjacent Bear watershed, the MF
American showed a surprisingly high degree of genetic differentiation
from other locations (Figure 2A). These data suggest that there is less
genetic differentiation between the NF American and the Bear watersheds,
than between the NF and MF American watersheds. We conclude that
measurements of overall genetic differentiation in FYLF from our study
area largely conform to watershed and geographic expectations, with the
exception of the American watershed, which shows a surprisingly high
degree of genetic differentiation between the North (unregulated) and
Middle (hydropeaking) Forks.

To further investigate patterns of genetic variation within the American
Watershed, we performed two PCAs, one on samples from the NF American,
and the other on samples from the MF American. The PCA of the NF
American showed minimal differentiation among locations, with different
study sites blending together and weak patterns of population structure
(Figure 2B). In contrast, PCA of the MF American showed strong
differentiation between sites (Figures 2C, 2D). The MF American PCA
completely resolved all sites, with the first component (PC1) strongly
differentiating the samples in the hydropeaking reach from all other
sites in the MF American. This pattern may be due to the differential
river regulation between the two rivers; the NF American is unregulated
and has weak PCA differentiation, whereas the MF American has a higher
level of river regulation and all sites form distinct genetic clusters,
indicative of reduced gene flow among sites within the MF American.
River regulation is the strongest predictor of genetic isolation with
FYLF in the Northern Sierra

To assess how patterns of genetic differentiation are associated with
river regulation across our entire study area, we estimated pairwise FST
(Wright, 1943) between all collection locations within a river for all
six rivers. We then plotted the scaled mean pairwise FST {[}mean FST /
(1-mean FST){]} (Rousset, 1997) for each location against the mean river
distance (the average distance along the river network from each
collection location to every other location within that study river).
Furthermore, each location was categorized by regulation level of
closest mainstem location (see Methods). While there was a clear
relationship between FST and river distance (as shown by the slope of
regression lines in Figure 3A), there was a striking pattern of elevated
FST by regulation type (Figure 3A). Even the bypass regulation type
showed a distinct pattern of elevated FST. For instance, regulated
rivers with locations separated by less than 10km had FST values
comparable to unregulated locations separated by mean river distances
over 30 km. Hydropeaking was the most extreme pattern of the three
regulation types and showed highly elevated FST values with the steepest
regression coefficient. The baseline FST or global mean increased by
over 0.1 between the unregulated (mean FST=0.141), and regulated
locations (global mean for bypass FST=0.256, hydropeaking FST=0.278).
This suggests a greater degree of isolation within sites in regulated
river reaches compared with FYLF populations in unregulated reaches as
larger FST values represent reductions in heterozygosity due to
population subdivision (Slatkin, 1987). We conclude FYLF in regulated
rivers show patterns of greater population isolation and loss of
heterozygosity compared to frogs in unregulated locations.

To investigate the relative influence of river regulation compared to
other covariates such as river distance on genetic differentiation
(i.e.~FST), we used boosted regression tree (BRT) modeling. Covariates
included flow regime alteration type, river distance, watershed
variables derived from National hydrology data (NHD), topographic data,
and allele frequency spectrum skew (see below, Methods). We found flow
regulation explained the greatest amount of variance in FST (Figure 3B).
Thus, river regulation has a larger relative influence than mean river
distance between sampling locations, which is often the most important
factor influencing genetic differentiation (Rousset, 1997; Slatkin,
1987; Wright, 1943). We conclude there is a pattern of isolation and
limited connectivity between populations in regulated reaches.

\textbf{\emph{River regulation strongly correlated with decreasing
genetic diversity in FYLF}} To investigate the association between river
regulation and genetic diversity trajectory (stable, increasing, or
decreasing), we summarized patterns of genetic variation using two
estimators of Œ∏ (4NŒº): Tajima's Œ∏ (Œ∏ùúã) is based on the average number of
pairwise differences (Tajima, 1983), and Watterson's Œ∏ (Œ∏S) is based on
the number of segregating sites (Watterson, 1975) (see methods). These
estimators are influenced by the demographic history of a population and
provide information on the trajectory of changes in genetic diversity.
When genetic diversity has been stable, these estimates should be equal;
when genetic diversity has been increasing, Œ∏ùúã \textgreater{} Œ∏S; and
when genetic diversity has been decreasing, Œ∏S \textgreater{} Œ∏ùúã. We
found zero populations sampled within regulated watersheds had evidence
of increasing genetic diversity (e.g., a Œ∏ùúã ‚àí Œ∏S that was less than
zero) (Figure 4A). The regulated locations showed a clear trajectory of
genetic diversity loss (Figure 4A, 4B). Three of the four hydropeaking
locations had the highest values of ŒîŒ∏ (Œ∏ùúã ‚àí Œ∏S), and the global mean
was significantly different from other regulation types. Although some
tributary populations within unregulated watersheds also showed signs of
genetic diversity loss, the mean genetic diversity trajectory at
unregulated locations was largely neutral (Figure 4B). This indicates
populations in the northern Sierra Nevada which are already limited in
number are losing genetic variation, and river regulation appears to be
exacerbating these patterns. We conclude there is evidence of recent
genetic diversity loss across populations in the regulated river
systems, regardless of regulation type.

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

Although massive parallel sequencing (MPS) technologies have the
potential to facilitate collection of high-quality genetic data in
virtually any species, a number of challenges still remain for many
species including low quality or non-existent reference genomes,
large/complex/repetitive genomes, and high cost of processing/sequencing
in studies with many samples. Amphibians are particularly challenging as
many species have very large genome sizes (Nunziata, Lance, Scott,
Lemmon, \& Weisrock, 2017) for example, there are only two frog
reference genome assemblies available as of 2018 (Hellsten et al., 2010;
Sun et al., 2015). Our results demonstrate that Rapture (Ali et al.,
2016) is a suitable method to rapidly and inexpensively discover a large
number of loci in a frog species with a complex genome. In this study,
we used new RAD sequencing and RAD capture (Rapture) methods (Ali et
al., 2016) to generate high-quality genomic data suitable for
discovering and genotyping many single nucleotide polymorphisms (SNPs)
in FYLF. Based on this dataset, we were able to successfully
characterize patterns of genetic variation within FYLF as well as design
a set of RAD capture baits that can be used as a genetic monitoring
resource for FYLF (and likely other ranid species). This highlights that
the collection of genetic information, even from large numbers of
samples or in complex genomes, is no longer a limitation with current
genomic methods such as RAD and Rapture.

Demographic connectivity is widely recognized as a fundamental driver of
long-term population persistence (Fahrig \& Merriam, 1985; Taylor,
Fahrig, Henein, \& Merriam, 1993). Populations must adapt over time and
connectivity is a major way to transfer genetic information. For
example, previous studies have shown that adaptation can occur by
spreading specific alleles across large geographic distances (Miller et
al., 2012; Prince et al., 2017). In many regulated river reaches in the
Sierra Nevada, FYLF now occur in isolated locations, breeding in
tributaries rather than mainstem habitats. However, since these frogs
have the potential to move long distances (FYLF have been observed
moving over 1 km per day (Bourque, 2008)), the extent to which current
population connectivity has been lost due to river regulation remains
unknown. Examining pairwise FST, revealed a major decrease in
connectivity in populations in regulated systems, even with limited
river regulation (i.e., bypass reaches). Usually isolation by distance
patterns best describe variation in genetic data, yet the primary factor
influencing genetic differentiation among these rivers is hydrologic
alteration (Figure 3B). Thus, despite being able to move long distances,
FYLF have not been able to maintain population connectivity in regulated
rivers. This demonstrates that even in species that can move relatively
long distances and pass potential physical barriers (e.g.,
infrastructure such as dams, canals, and reservoirs likely do not
represent barriers to movement of FYLF) loss of connectivity may still
occur and can be revealed with genetic analysis.

Genetic diversity is also a critical component for long-term population
persistence because it is closely related to the evolutionary capacity
for adaptation to environmental changes (Frankham, 2002; Hoffmann \&
Sgr√≤, 2011; Ishiyama, Koizumi, Yuta, \& Nakamura, 2015; Lande \&
Shannon, 1996). In some cases, isolated populations can maintain genetic
diversity when they are sufficiently sized (Whiteley et al., 2010),
however, species of conservation concern typically have small and/or
declining populations and thus may be susceptible to genetic diversity
loss (Frankham, 2002; Krohn et al., 2018). Throughout the Sierra Nevada,
FYLF have largely disappeared from regulated mainstem rivers, but the
extent to which existing populations have been able to maintain genetic
diversity is unclear. Strikingly, our analysis revealed that every
single population within the regulated watersheds exhibits a trajectory
of genetic diversity loss. Thus, genomic analysis of molecular variation
provides a powerful lens to discover and assess trajectories of genetic
diversity.

Our analyses, using metrics that serve as a reasonable proxy for genetic
health, does not bode well for the long-term persistence of FYLF
populations in regulated rivers in the Sierra Nevada. Many of these FYLF
populations are already losing genetic diversity and given their small
size and reduced connectivity the effects of inbreeding will likely
exacerbate their problems. FYLF have evolved in river systems with
consistent hydrologic seasonality and predictability, despite
inter-annual variation. Flow regulation has altered patterns of
hydrologic seasonality and predictability in many watersheds (Sarah J.
Kupferberg et al., 2012). Long-term population persistence may still be
possible if conservation efforts utilize methods that promote or
maintain genetic health and increase population connectivity. For
example, simulations by Botero et al. (2015) demonstrated adaptation
persisted in modeled populations through large environmental
changes---if phenotypic strategies were appropriate before and after the
change---but modeled populations declined rapidly when the current
strategy was a mismatch to the current environment. Thus, FYLF
conservation efforts should focus on river reaches where flow management
may provide opportunities to more closely mimic local natural flow
regimes and thus improve hydrologic connectivity.

Detecting evolutionary responses to within- and among-year changes in an
ecological or hydrological context has previously been difficult.
However, utilizing genetic data can fill these gaps and provide a highly
informative process for identifying the impacts of anthropogenic and
environmental change on the process of adaptation (Botero et al., 2015;
Kahilainen, Puurtinen, \& Kotiaho, 2014). We demonstrate that an aquatic
species that has adapted to local hydrology patterns shows poor genetic
health (i.e., clear patterns of decreased connectivity and trajectories
of genetic diversity loss). Our results highlight the potential impact
of river regulation on aquatic organisms and their potential for long
term persistence. In the future, similar genetic approaches could be
used in many other contexts to explore the impacts of river regulation
on aquatic organisms. Taken together, our results demonstrate that
genetic monitoring can be a powerful tool for assessment of population
health and should be a critical component of conservation management in
aquatic organisms.

\textbf{Data Archiving Statement:} Should the manuscript be accepted,
the data supporting the results will be archived in an appropriate
public repository such as Dryad, and the data DOI will be appended to
the end of the article.

\texttt{\{r\ maxdelays,\ results="asis",\ echo=Fevalrary(knitr)\ kable(max\_delays,\ \ \ \ \ \ \ \ col.names\ =\ c("Airline",\ "Max\ Arrival\ Delay"),\ \ \ \ \ \ \ caption\ =\ "Maximum\ Delays\ by\ Airline",\ \ \ \ \ \ \ caption.short\ =\ "Max\ Delays\ by\ Airline",\ \ \ \ \ \ \ longtable\ =\ TRUE,\ \ \ \ \ \ \ booktabs\ =\ TRUE)}

The last two options make the table a little easier-to-read.

\hypertarget{hybrids}{%
\chapter{Rare hybridization detected between two sympatric Ranid frog
species in the northern Sierra Nevada, California}\label{hybrids}}

\hypertarget{introduction-2}{%
\section{Introduction}\label{introduction-2}}

Landscape changes can influence species demography and migration
patterns (Li et al.~2017) by changing rates of gene flow within species.
Changing migration rates and population sizes can influence population
structure, thus over time, landscape changes can affect significant
change in a species. Furthermore, cross-breeding or hybridization
between closely related taxa can promote gene flow (introgression)
between species, which may be an important evolutionary mechanism for
either homogenization (reversing initial divergence between species) or
reproductive isolation (potentially leading to speciation)(Mallet 2007,
Barrera-Guzm√°n et al.~2018). Hybridization events in vertebrates may be
rare, or rarely detected, and thus identifying potential hybridization
can be difficult and may be affected by sampling design, timing, and
resolution of genetic markers. Therefore, occurrences of hybridization
likely remain unknown, particularly in cryptic taxa. Assessing
population admixture or detecting potential hybridization has previously
been challenging; however, modern genetic methods provide a powerful
approach to assess populations at fine geographic and evolutionary
scales(Ali et al.~2016, Prince et al.~2017). We investigate the
potential for hybridization in two sympatrically occurring endemic frog
species in the Sierra Nevada of California, USA. Foothill yellow-legged
frogs, Rana boylii, (Baird 1856) historically occurred in lower and
mid-elevation (\textless{} 1500 m) streams and rivers from Southern
Oregon to northern Baja California west of the Sierra-Cascade crest
(Stebbins 2003), whereas Sierra Nevada yellow-legged frogs, Rana
sierrae, (Camp 1917) typically occur from 1500 m to over 3600 m in lakes
and streams (Stebbins 2003). Population declines have been documented
across the former range of both of these species; R. sierrae has been
extirpated from over 90 percent of its historical range (Drost and
Fellers 1996, Vredenburg et al.~2007) while R. boylii has been
extirpated from 50 percent of its historical range (Jennings and Hayes
1994, Davidson et al.~2002). Both species are species of conservation
concern; in 2014, the U. S. Fish and Wildlife Service (USFWS) listed R.
sierrae as endangered under the U. S. Endangered Species Act (ESA)(USFWS
2014), and R. boylii is a state species of special concern and a
candidate for listing under state and federal ESA. Unlike other ranid
frog species with broad areas of potential intergradation (Shaffer et
al.~2004), R. boylii and R. sierrae only rarely occur sympatrically.
Zweifel (1955) described one historical location where these two species
co-occurred, in Butte County near DeSabla. Currently the only known
locations where both species are found are in several tributaries to the
Feather River in the northern Sierra Nevada, California (Figure 1).
Hybridization between these species has not previously been successfully
documented---breeding experiments by Zwiefel (1955) between R. sierrae
(formerly known as R. muscosa) and R. boylii yielded very low viability
and high incidences of embryological abnormalities---likely indicating a
post-zygotic barrier between the species. However, these experiments
only crossed female R. sierrae individuals with male R. boylii, and the
individuals were from very different California regions (e.g., Butte and
Nevada County vs.~Contra Costa County). R. boylii and R. sierrae can
have very similar morphology and habitat preferences in areas they
co-occur, thus assigning individuals to species is difficult and
imprecise using field identification methods. This presents a challenge
for management because these sympatric species have different
conservation status and management objectives. In cases where
hybridization is present, conservation actions must consider whether
hybridization may lead to reproductive isolation (Barrera-Guzm√°n et
al.~2018b). We employed modern genetic methodology to better understand
the evolutionary history of R. sierrae and R. boylii where their ranges
overlap. We investigated three primary questions:
\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Can hybridization be detected between two sympatrically occurring
  threatened and endangered (ESA) frog species in the Sierra Nevada
  using data generated from genome-wide single nucleotide polymorphisms
  (SNPs);
\item
  If hybrids can be detected, do genetic signatures suggest hybrid
  viability;
\item
  Using coalescent modeling, are migration rates between species in
  sympatrically occurring populations higher than in allopatrically
  occurring populations in adjacent watersheds?
\end{enumerate}
\hypertarget{chemistry-101-symbols}{%
\section{Chemistry 101: Symbols}\label{chemistry-101-symbols}}

Chemical formulas will look best if they are not italicized. Get around
math mode's automatic italicizing in LaTeX by using the argument
\texttt{\$\textbackslash{}mathrm\{formula\ here\}\$}, with your formula
inside the curly brackets. (Notice the use of the backticks here which
enclose text that acts as code.)

So, \(\mathrm{Fe_2^{2+}Cr_2O_4}\) is written
\texttt{\$\textbackslash{}mathrm\{Fe\_2\^{}\{2+\}Cr\_2O\_4\}\$}.

\noindent Exponent or Superscript: \(\mathrm{O^-}\)

\noindent Subscript: \(\mathrm{CH_4}\)

To stack numbers or letters as in \(\mathrm{Fe_2^{2+}}\), the subscript
is defined first, and then the superscript is defined.

\noindent Bullet: CuCl \(\bullet\) \(\mathrm{7H_{2}O}\)

\noindent Delta: \(\Delta\)

\noindent Reaction Arrows: \(\longrightarrow\) or
\(\xrightarrow{solution}\)

\noindent Resonance Arrows: \(\leftrightarrow\)

\noindent Reversible Reaction Arrows: \(\rightleftharpoons\)

\hypertarget{typesetting-reactions}{%
\subsection{Typesetting reactions}\label{typesetting-reactions}}

You may wish to put your reaction in an equation environment, which
means that LaTeX will place the reaction where it fits and will number
the equations for you.
\begin{equation}
  \mathrm{C_6H_{12}O_6  + 6O_2} \longrightarrow \mathrm{6CO_2 + 6H_2O}
  \label{eq:reaction}
\end{equation}
We can reference this combustion of glucose reaction via Equation
\eqref{eq:reaction}.

\hypertarget{other-examples-of-reactions}{%
\subsection{Other examples of
reactions}\label{other-examples-of-reactions}}

\(\mathrm{NH_4Cl_{(s)}}\) \(\rightleftharpoons\)
\(\mathrm{NH_{3(g)}+HCl_{(g)}}\)

\noindent \(\mathrm{MeCH_2Br + Mg}\) \(\xrightarrow[below]{above}\)
\(\mathrm{MeCH_2\bullet Mg \bullet Br}\)

\hypertarget{physics}{%
\section{Physics}\label{physics}}

Many of the symbols you will need can be found on the math page
\url{http://web.reed.edu/cis/help/latex/math.html} and the Comprehensive
LaTeX Symbol Guide
(\url{http://mirror.utexas.edu/ctan/info/symbols/comprehensive/symbols-letter.pdf}).

\hypertarget{biology}{%
\section{Biology}\label{biology}}

You will probably find the resources at
\url{http://www.lecb.ncifcrf.gov/~toms/latex.html} helpful, particularly
the links to bsts for various journals. You may also be interested in
TeXShade for nucleotide typesetting
(\url{http://homepages.uni-tuebingen.de/beitz/txe.html}). Be sure to
read the proceeding chapter on graphics and tables.

\hypertarget{ref-labels}{%
\chapter{Tables, Graphics, References, and Labels}\label{ref-labels}}

\hypertarget{tables}{%
\section{Tables}\label{tables}}

By far the easiest way to present tables in your thesis is to store the
contents of the table in a CSV or Excel file, then read that file in to
your R Markdown document as a data frame. Then you can style the table
with the \texttt{kable} function, or functions in the
\href{https://cran.r-project.org/web/packages/kableExtra/index.html}{kableExtra}
pacakge.

In addition to the tables that can be automatically generated from a
data frame in \textbf{R} that you saw in {[}R Markdown Basics{]} using
the \texttt{kable} function, you can also create tables using
\emph{pandoc}. (More information is available at
\url{http://pandoc.org/README.html\#tables}.) This might be useful if
you don't have values specifically stored in \textbf{R}, but you'd like
to display them in table form. Below is an example. Pay careful
attention to the alignment in the table and hyphens to create the rows
and columns. Generally I don't recommend this approach of typing the
table directly into your R Markdown document.
\begin{longtable}[]{@{}ccc@{}}
\caption{\label{tab:inher} Correlation of Inheritance Factors for Parents
and Child}\tabularnewline
\toprule
\begin{minipage}[b]{0.29\columnwidth}\centering
Factors\strut
\end{minipage} & \begin{minipage}[b]{0.46\columnwidth}\centering
Correlation between Parents \& Child\strut
\end{minipage} & \begin{minipage}[b]{0.16\columnwidth}\centering
Inherited\strut
\end{minipage}\tabularnewline
\midrule
\endfirsthead
\toprule
\begin{minipage}[b]{0.29\columnwidth}\centering
Factors\strut
\end{minipage} & \begin{minipage}[b]{0.46\columnwidth}\centering
Correlation between Parents \& Child\strut
\end{minipage} & \begin{minipage}[b]{0.16\columnwidth}\centering
Inherited\strut
\end{minipage}\tabularnewline
\midrule
\endhead
\begin{minipage}[t]{0.29\columnwidth}\centering
Education\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\centering
-0.49\strut
\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\centering
Yes\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.29\columnwidth}\centering
Socio-Economic Status\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\centering
0.28\strut
\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\centering
Slight\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.29\columnwidth}\centering
Income\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\centering
0.08\strut
\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\centering
No\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.29\columnwidth}\centering
Family Size\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\centering
0.18\strut
\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\centering
Slight\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.29\columnwidth}\centering
Occupational Prestige\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\centering
0.21\strut
\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\centering
Slight\strut
\end{minipage}\tabularnewline
\bottomrule
\end{longtable}
We can also create a link to the table by doing the following: Table
\ref{tab:inher}. If you go back to {[}Loading and exploring data{]} and
look at the \texttt{kable} table, we can create a reference to this max
delays table too: Table \ref{tab:maxdelays}. The addition of the
\texttt{(\textbackslash{}\#tab:inher)} option to the end of the table
caption allows us to then make a reference to Table
\texttt{\textbackslash{}@ref(tab:label)}. Note that this reference could
appear anywhere throughout the document after the table has appeared.

\clearpage

\hypertarget{figures}{%
\section{Figures}\label{figures}}

If your thesis has a lot of figures, \emph{R Markdown} might behave
better for you than that other word processor. One perk is that it will
automatically number the figures accordingly in each chapter. You'll
also be able to create a label for each figure, add a caption, and then
reference the figure in a way similar to what we saw with tables
earlier. If you label your figures, you can move the figures around and
\emph{R Markdown} will automatically adjust the numbering for you. No
need for you to remember! So that you don't have to get too far into
LaTeX to do this, a couple \textbf{R} functions have been created for
you to assist. You'll see their use below.

In the \textbf{R} chunk below, we will load in a picture stored as
\texttt{uw.png} in our main directory. We then give it the caption of
``UW logo'', the label of ``uwlogo'', and specify that this is a figure.
Make note of the different \textbf{R} chunk options that are given in
the R Markdown file (not shown in the knitted document).
\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{#include_graphics(path = "figure/uw.png")}
\end{Highlighting}
\end{Shaded}
Here is a reference to the UW logo: Figure \ref{fig:uwlogo}. Note the
use of the \texttt{fig:} code here. By naming the \textbf{R} chunk that
contains the figure, we can then reference that figure later as done in
the first sentence here. We can also specify the caption for the figure
via the R chunk option \texttt{fig.cap}.

\clearpage

Below we will investigate how to save the output of an \textbf{R} plot
and label it in a way similar to that done above. Recall the
\texttt{flights} dataset from Chapter \ref{rmd-basics}. (Note that we've
shown a different way to reference a section or chapter here.) We will
next explore a bar graph with the mean flight departure delays by
airline from Portland for 2014. Note also the use of the \texttt{scale}
parameter which is discussed on the next page.
\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{flights }\OperatorTok{%>%}\StringTok{ }\KeywordTok{group_by}\NormalTok{(carrier) }\OperatorTok{%>%}
\StringTok{  }\KeywordTok{summarize}\NormalTok{(}\DataTypeTok{mean_dep_delay =} \KeywordTok{mean}\NormalTok{(dep_delay)) }\OperatorTok{%>%}
\StringTok{  }\KeywordTok{ggplot}\NormalTok{(}\KeywordTok{aes}\NormalTok{(}\DataTypeTok{x =}\NormalTok{ carrier, }\DataTypeTok{y =}\NormalTok{ mean_dep_delay)) }\OperatorTok{+}
\StringTok{  }\KeywordTok{geom_bar}\NormalTok{(}\DataTypeTok{position =} \StringTok{"identity"}\NormalTok{, }\DataTypeTok{stat =} \StringTok{"identity"}\NormalTok{, }\DataTypeTok{fill =} \StringTok{"red"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}
Here is a reference to this image: Figure \ref{fig:delaysboxplot}.

A table linking these carrier codes to airline names is available at
\url{https://github.com/ismayc/pnwflights14/blob/master/data/airlines.csv}.

\clearpage

Next, we will explore the use of the \texttt{out.extra} chunk option,
which can be used to shrink or expand an image loaded from a file by
specifying \texttt{"scale=\ "}. Here we use the mathematical graph
stored in the ``subdivision.pdf'' file.
\begin{figure}
\includegraphics[scale=0.75]{figure/subdivision} \caption{Subdiv. graph}\label{fig:subd}
\end{figure}
Here is a reference to this image: Figure \ref{fig:subd}. Note that
\texttt{echo=FALSE} is specified so that the \textbf{R} code is hidden
in the document.

\textbf{More Figure Stuff}

Lastly, we will explore how to rotate and enlarge figures using the
\texttt{out.extra} chunk option. (Currently this only works in the PDF
version of the book.)
\begin{figure}
\includegraphics[angle=180, scale=1.1]{figure/subdivision} \caption{A Larger Figure, Flipped Upside Down}\label{fig:subd2}
\end{figure}
As another example, here is a reference: Figure \ref{fig:subd2}.

\hypertarget{footnotes-and-endnotes}{%
\section{Footnotes and Endnotes}\label{footnotes-and-endnotes}}

You might want to footnote something.\footnote{footnote text} The
footnote will be in a smaller font and placed appropriately. Endnotes
work in much the same way.

\hypertarget{bibliographies}{%
\section{Bibliographies}\label{bibliographies}}

Of course you will need to cite things, and you will probably accumulate
an armful of sources. There are a variety of tools available for
creating a bibliography database (stored with the .bib extension). In
addition to BibTeX suggested below, you may want to consider using the
free and easy-to-use tool called Zotero. Some Zotero documentation is at
\url{http://libguides.reed.edu/citation/zotero}. In addition, a tutorial
is available from Middlebury College at
\url{http://sites.middlebury.edu/zoteromiddlebury/}.

\emph{R Markdown} uses \emph{pandoc} (\url{http://pandoc.org/}) to build
its bibliographies. One nice caveat of this is that you won't have to do
a second compile to load in references as standard LaTeX requires. To
cite references in your thesis (after creating your bibliography
database), place the reference name inside square brackets and precede
it by the ``at'' symbol. For example, here's a reference to a book about
worrying: ({\textbf{???}}). This \texttt{Molina1994} entry appears in a
file called \texttt{thesis.bib} in the \texttt{bib} folder. This
bibliography database file was created by a program called BibTeX. You
can call this file something else if you like (look at the YAML header
in the main .Rmd file) and, by default, is to placed in the \texttt{bib}
folder.

For more information about BibTeX and bibliographies, see
(\url{http://web.reed.edu/cis/help/latex/index.html})\footnote{({\textbf{???}})}.
There are three pages on this topic: \emph{bibtex} (which talks about
using BibTeX, at \url{http://web.reed.edu/cis/help/latex/bibtex.html}),
\emph{bibtexstyles} (about how to find and use the bibliography style
that best suits your needs, at
\url{http://web.reed.edu/cis/help/latex/bibtexstyles.html}) and
\emph{bibman} (which covers how to make and maintain a bibliography by
hand, without BibTeX, at
\url{http://web.reed.edu/cis/help/latex/bibman.html}). The last page
will not be useful unless you have only a few sources.

If you look at the YAML header at the top of the main .Rmd file you can
see that we can specify the style of the bibliography by referencing the
appropriate csl file. You can download a variety of different style
files at \url{https://www.zotero.org/styles}. Make sure to download the
file into the csl folder.

\textbf{Tips for Bibliographies}
\begin{itemize}
\tightlist
\item
  Like with thesis formatting, the sooner you start compiling your
  bibliography for something as large as thesis, the better.
\item
  The cite key (a citation's label) needs to be unique from the other
  entries.
\item
  When you have more than one author or editor, you need to separate
  each author's name by the word ``and'' e.g.
  \texttt{Author\ =\ \{Noble,\ Sam\ and\ Youngberg,\ Jessica\},}.
\item
  Bibliographies made using BibTeX (whether manually or using a manager)
  accept LaTeX markup, so you can italicize and add symbols as
  necessary.
\item
  To force capitalization in an article title or where all lowercase is
  generally used, bracket the capital letter in curly braces.
\end{itemize}
\hypertarget{anything-else}{%
\section{Anything else?}\label{anything-else}}

If you'd like to see examples of other things in this template, please
\href{https://github.com/benmarwick/huskydown/issues/new}{contact us}
(email \href{mailto:bmarwick@uw.edu}{\nolinkurl{bmarwick@uw.edu}}) with
your suggestions. We love to see people using \emph{R Markdown} for
their theses, and are happy to help.

\hypertarget{conclusion}{%
\chapter*{Conclusion}\label{conclusion}}
\addcontentsline{toc}{chapter}{Conclusion}

If we don't want Conclusion to have a chapter number next to it, we can
add the \texttt{\{-\}} attribute.

\textbf{More info}

And here's some other random info: the first paragraph after a chapter
title or section head \emph{shouldn't be} indented, because indents are
to tell the reader that you're starting a new paragraph. Since that's
obvious after a chapter or section title, proper typesetting doesn't add
an indent there.

\appendix

\hypertarget{the-first-appendix}{%
\chapter{The First Appendix}\label{the-first-appendix}}

This first appendix includes all of the R chunks of code that were
hidden throughout the document (using the \texttt{include\ =\ FALSE}
chunk tag) to help with readibility and/or setup.

\textbf{In the main Rmd file}
\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{# This chunk ensures that the huskydown package is}
\CommentTok{# installed and loaded. This huskydown package includes}
\CommentTok{# the template files for the thesis.}
\ControlFlowTok{if}\NormalTok{(}\OperatorTok{!}\KeywordTok{require}\NormalTok{(devtools))}
  \KeywordTok{install.packages}\NormalTok{(}\StringTok{"devtools"}\NormalTok{, }\DataTypeTok{repos =} \StringTok{"http://cran.rstudio.com"}\NormalTok{)}
\ControlFlowTok{if}\NormalTok{(}\OperatorTok{!}\KeywordTok{require}\NormalTok{(huskydown))}
\NormalTok{  devtools}\OperatorTok{::}\KeywordTok{install_github}\NormalTok{(}\StringTok{"benmarwick/huskydown"}\NormalTok{)}
\KeywordTok{library}\NormalTok{(huskydown)}
\CommentTok{#if(!require(huskydown))}
\CommentTok{#  devtools::install_github("danovando/gauchodown")}
\CommentTok{#library(gauchodown)}
\end{Highlighting}
\end{Shaded}
\textbf{In Chapter \ref{ref-labels}:}
\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{# This chunk ensures that the huskydown package is}
\CommentTok{# installed and loaded. This huskydown package includes}
\CommentTok{# the template files for the thesis and also two functions}
\CommentTok{# used for labeling and referencing}
\ControlFlowTok{if}\NormalTok{(}\OperatorTok{!}\KeywordTok{require}\NormalTok{(devtools))}
  \KeywordTok{install.packages}\NormalTok{(}\StringTok{"devtools"}\NormalTok{, }\DataTypeTok{repos =} \StringTok{"http://cran.rstudio.com"}\NormalTok{)}
\ControlFlowTok{if}\NormalTok{(}\OperatorTok{!}\KeywordTok{require}\NormalTok{(dplyr))}
    \KeywordTok{install.packages}\NormalTok{(}\StringTok{"dplyr"}\NormalTok{, }\DataTypeTok{repos =} \StringTok{"http://cran.rstudio.com"}\NormalTok{)}
\ControlFlowTok{if}\NormalTok{(}\OperatorTok{!}\KeywordTok{require}\NormalTok{(ggplot2))}
    \KeywordTok{install.packages}\NormalTok{(}\StringTok{"ggplot2"}\NormalTok{, }\DataTypeTok{repos =} \StringTok{"http://cran.rstudio.com"}\NormalTok{)}
\ControlFlowTok{if}\NormalTok{(}\OperatorTok{!}\KeywordTok{require}\NormalTok{(ggplot2))}
    \KeywordTok{install.packages}\NormalTok{(}\StringTok{"bookdown"}\NormalTok{, }\DataTypeTok{repos =} \StringTok{"http://cran.rstudio.com"}\NormalTok{)}
\ControlFlowTok{if}\NormalTok{(}\OperatorTok{!}\KeywordTok{require}\NormalTok{(huskydown))\{}
  \KeywordTok{library}\NormalTok{(devtools)}
\NormalTok{  devtools}\OperatorTok{::}\KeywordTok{install_github}\NormalTok{(}\StringTok{"benmarwick/huskydown"}\NormalTok{)}
\NormalTok{  \}}
\KeywordTok{library}\NormalTok{(huskydown)}
\KeywordTok{library}\NormalTok{(knitr)}
\NormalTok{flights <-}\StringTok{ }\KeywordTok{read.csv}\NormalTok{(}\StringTok{"data/flights.csv"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}
\hypertarget{the-second-appendix-for-fun}{%
\chapter{The Second Appendix, for
Fun}\label{the-second-appendix-for-fun}}

\hypertarget{colophon}{%
\chapter*{Colophon}\label{colophon}}
\addcontentsline{toc}{chapter}{Colophon}

This document is set in \href{https://github.com/georgd/EB-Garamond}{EB
Garamond}, \href{https://github.com/adobe-fonts/source-code-pro/}{Source
Code Pro} and \href{http://www.latofonts.com/lato-free-fonts/}{Lato}.
The body text is set at 11pt with \(\familydefault\).

It was written in R Markdown and \(\LaTeX\), and rendered into PDF using
\href{https://github.com/benmarwick/huskydown}{huskydown} and
\href{https://github.com/rstudio/bookdown}{bookdown}.

This document was typeset using the XeTeX typesetting system, and the
\href{http://staff.washington.edu/fox/tex/}{University of Washington
Thesis class} class created by Jim Fox. Under the hood, the
\href{https://github.com/UWIT-IAM/UWThesis}{University of Washington
Thesis LaTeX template} is used to ensure that documents conform
precisely to submission standards. Other elements of the document
formatting source code have been taken from the
\href{https://github.com/stevenpollack/ucbthesis}{Latex, Knitr, and
RMarkdown templates for UC Berkeley's graduate thesis}, and
\href{https://github.com/suchow/Dissertate}{Dissertate: a LaTeX
dissertation template to support the production and typesetting of a PhD
dissertation at Harvard, Princeton, and NYU}

The source files for this thesis, along with all the data files, have
been organised into an R package, xxx, which is available at
\url{https://github.com/xxx/xxx}. A hard copy of the thesis can be found
in the University of Washington library.

This version of the thesis was generated on 2018-08-30 09:58:55. The
repository is currently at this commit:

The computational environment that was used to generate this version is
as follows:
\begin{verbatim}
Session info -------------------------------------------------------------
\end{verbatim}
\begin{verbatim}
 setting  value                       
 version  R version 3.5.1 (2018-07-02)
 system   x86_64, darwin15.6.0        
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2018-08-30                  
\end{verbatim}
\begin{verbatim}
Packages -----------------------------------------------------------------
\end{verbatim}
\begin{verbatim}
 package    * version    date       source                               
 assertthat   0.2.0      2017-04-11 CRAN (R 3.5.0)                       
 backports    1.1.2      2017-12-13 CRAN (R 3.5.0)                       
 base       * 3.5.1      2018-07-05 local                                
 bindr        0.1.1      2018-03-13 CRAN (R 3.5.0)                       
 bindrcpp     0.2.2      2018-03-29 CRAN (R 3.5.0)                       
 bookdown     0.7        2018-02-18 CRAN (R 3.5.0)                       
 colorspace   1.3-2      2016-12-14 CRAN (R 3.5.0)                       
 compiler     3.5.1      2018-07-05 local                                
 crayon       1.3.4      2017-09-16 CRAN (R 3.5.0)                       
 datasets   * 3.5.1      2018-07-05 local                                
 devtools   * 1.13.6     2018-06-27 CRAN (R 3.5.0)                       
 digest       0.6.16     2018-08-22 CRAN (R 3.5.1)                       
 dplyr      * 0.7.6      2018-06-29 CRAN (R 3.5.0)                       
 evaluate     0.11       2018-07-17 CRAN (R 3.5.0)                       
 ggplot2    * 3.0.0.9000 2018-08-29 Github (tidyverse/ggplot2@71cb174)   
 git2r        0.23.0     2018-07-17 CRAN (R 3.5.0)                       
 glue         1.3.0      2018-07-17 CRAN (R 3.5.0)                       
 graphics   * 3.5.1      2018-07-05 local                                
 grDevices  * 3.5.1      2018-07-05 local                                
 grid         3.5.1      2018-07-05 local                                
 gtable       0.2.0      2016-02-26 CRAN (R 3.5.0)                       
 htmltools    0.3.6      2017-04-28 CRAN (R 3.5.0)                       
 huskydown  * 0.0.5      2018-07-30 Github (danovando/gauchodown@5a18965)
 knitr      * 1.20       2018-02-20 CRAN (R 3.5.0)                       
 lazyeval     0.2.1      2017-10-29 CRAN (R 3.5.0)                       
 magrittr     1.5        2014-11-22 CRAN (R 3.5.0)                       
 memoise      1.1.0      2017-04-21 CRAN (R 3.5.0)                       
 methods    * 3.5.1      2018-07-05 local                                
 munsell      0.5.0      2018-06-12 CRAN (R 3.5.0)                       
 pillar       1.3.0      2018-07-14 CRAN (R 3.5.0)                       
 pkgconfig    2.0.2      2018-08-16 CRAN (R 3.5.0)                       
 plyr         1.8.4      2016-06-08 CRAN (R 3.5.0)                       
 purrr        0.2.5      2018-05-29 CRAN (R 3.5.0)                       
 R6           2.2.2      2017-06-17 CRAN (R 3.5.0)                       
 Rcpp         0.12.18    2018-07-23 CRAN (R 3.5.1)                       
 rlang        0.2.2      2018-08-16 CRAN (R 3.5.0)                       
 rmarkdown    1.10       2018-06-11 cran (@1.10)                         
 rprojroot    1.3-2      2018-01-03 CRAN (R 3.5.0)                       
 rstudioapi   0.7        2017-09-07 CRAN (R 3.5.0)                       
 scales       1.0.0.9000 2018-08-29 Github (hadley/scales@0f7a186)       
 stats      * 3.5.1      2018-07-05 local                                
 stringi      1.2.4      2018-07-20 CRAN (R 3.5.0)                       
 stringr      1.3.1      2018-05-10 CRAN (R 3.5.0)                       
 tibble       1.4.2      2018-01-22 CRAN (R 3.5.0)                       
 tidyselect   0.2.4      2018-02-26 CRAN (R 3.5.0)                       
 tools        3.5.1      2018-07-05 local                                
 utils      * 3.5.1      2018-07-05 local                                
 withr        2.1.2      2018-08-29 Github (jimhester/withr@8b9cee2)     
 xfun         0.3        2018-07-06 CRAN (R 3.5.0)                       
 yaml         2.2.0      2018-07-25 CRAN (R 3.5.0)                       
\end{verbatim}
\backmatter

\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

\markboth{References}{References}

\noindent

\setlength{\parindent}{-0.20in}
\setlength{\leftskip}{0.20in}
\setlength{\parskip}{8pt}

\hypertarget{refs}{}
\leavevmode\hypertarget{ref-bunn_basic_2002}{}%
Bunn, S. E., and A. H. Arthington. 2002. Basic principles and ecological
consequences of altered flow regimes for aquatic biodiversity.
Environmental management 30:492--507.

\leavevmode\hypertarget{ref-caze_could_2016}{}%
Caz√©, A. L. R., G. M√§der, T. S. Nunes, L. P. Queiroz, G. de Oliveira, J.
A. F. Diniz-Filho, S. L. Bonatto, and L. B. Freitas. 2016. Could refuge
theory and rivers acting as barriers explain the genetic variability
distribution in the Atlantic Forest? Molecular phylogenetics and
evolution 101:242--251.

\leavevmode\hypertarget{ref-dudgeon_freshwater_2006}{}%
Dudgeon, D., A. H. Arthington, M. O. Gessner, Z.-I. Kawabata, D. J.
Knowler, C. L√©v√™que, R. J. Naiman, A.-H. Prieur-Richard, D. Soto, M. L.
J. Stiassny, and C. A. Sullivan. 2006. Freshwater biodiversity:
Importance, threats, status and conservation challenges. Biological
reviews of the Cambridge Philosophical Society 81:163--182.

\leavevmode\hypertarget{ref-moyle_rapid_2011}{}%
Moyle, P. B., J. V. E. Katz, and R. M. Qui√±ones. 2011. Rapid decline of
California's native inland fishes: A status assessment. Biological
conservation 144:2414--2423.

\leavevmode\hypertarget{ref-murchie_fish_2008}{}%
Murchie, K. J., K. P. E. Hair, C. E. Pullen, T. D. Redpath, H. R.
Stephens, and S. J. Cooke. 2008. Fish response to modified flow regimes
in regulated rivers: Research methods, effects and opportunities. River
research and applications 24:197--217.

\leavevmode\hypertarget{ref-nilsson_fragmentation_2005}{}%
Nilsson, C., C. A. Reidy, M. Dynesius, and C. Revenga. 2005.
Fragmentation and flow regulation of the world's large river systems.
Science 308:405--408.

\leavevmode\hypertarget{ref-poff_homogenization_2007}{}%
Poff, N. L., J. D. Olden, D. M. Merritt, and D. M. Pepin. 2007.
Homogenization of regional river dynamics by dams and global
biodiversity implications. Proceedings of the National Academy of
Sciences of the United States of America 104:5732--5737.

\leavevmode\hypertarget{ref-power_dams_1996}{}%
Power, M. E., W. E. Dietrich, and J. C. Finlay. 1996. Dams and
Downstream Aquatic Biodiversity: Potential Food Web Consequences of
Hydrologic and Geomorphic Change. Environmental management 20:887--895.

\leavevmode\hypertarget{ref-pringle_what_2003}{}%
Pringle, C. 2003. What is hydrologic connectivity and why is it
ecologically important? Hydrological processes 17:2685--2689.

\leavevmode\hypertarget{ref-pringle_hydrologic_2001}{}%
Pringle, C. M. 2001. Hydrologic Connectivity and the Management of
Biological Reserves: A Global Perspective. Ecological applications: a
publication of the Ecological Society of America 11:981--998.

\leavevmode\hypertarget{ref-schick_directed_2007}{}%
Schick, R. S., and S. T. Lindley. 2007. Directed connectivity among fish
populations in a riverine network. The Journal of applied ecology
44:1116--1126.

\leavevmode\hypertarget{ref-shaw_importance_2016}{}%
Shaw, E. A., E. Lange, J. D. Shucksmith, and D. N. Lerner. 2016.
Importance of partial barriers and temporal variation in flow when
modelling connectivity in fragmented river systems. Ecological
engineering 91:515--528.

\leavevmode\hypertarget{ref-tonkin_seasonality_2017}{}%
Tonkin, J. D., M. T. Bogan, N. Bonada, B. Rios-Touma, and D. A. Lytle.
2017. Seasonality and predictability shape temporal species diversity.
Ecology 98:1201--1216.

\leavevmode\hypertarget{ref-voelker_river_2013}{}%
Voelker, G., B. D. Marks, C. Kahindo, U. A'genonga, F. Bapeamoni, L. E.
Duffie, J. W. Huntley, E. Mulotwa, S. A. Rosenbaum, and J. E. Light.
2013. River barriers and cryptic biodiversity in an evolutionary museum.
Ecology and evolution 3:536--545.

\leavevmode\hypertarget{ref-wiens_riverine_2002}{}%
Wiens, J. A. 2002. Riverine landscapes: Taking landscape ecology into
the water. Freshwater biology 47:501--515.

\leavevmode\hypertarget{ref-yarnell_ecology_2010}{}%
Yarnell, S. M., J. H. Viers, and J. F. Mount. 2010. Ecology and
Management of the Spring Snowmelt Recession. Bioscience 60:114--127.

\end{ucmainmatter}
\end{document}

%---Set Headers and Footers ------------------------------------------------------
\pagestyle{fancy}
\renewcommand{\chaptermark}[1]{\markboth{{\sf #1 \hspace*{\fill} Chapter~\thechapter}}{} }
\renewcommand{\sectionmark}[1]{\markright{ {\sf Section~\thesection \hspace*{\fill} #1 }}}
\fancyhf{}

\makeatletter \if@twoside \fancyhead[LO]{\small \rightmark} \fancyhead[RE]{\small\leftmark} \else \fancyhead[LO]{\small\leftmark}
\fancyhead[RE]{\small\rightmark} \fi

\def\cleardoublepage{\clearpage\if@openright \ifodd\c@page\else
  \hbox{}
  \vspace*{\fill}
  \begin{center}
    This page intentionally left blank
  \end{center}
  \vspace{\fill}
  \thispagestyle{plain}
  \newpage
  \fi \fi}
\makeatother
\fancyfoot[c]{\textrm{\textup{\thepage}}} % page number
\fancyfoot[C]{\thepage}
\renewcommand{\headrulewidth}{0.4pt}

\fancypagestyle{plain} { \fancyhf{} \fancyfoot[C]{\thepage}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}}
