%%%---PREAMBLE---%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\documentclass[twoside,12pt,final]{ucthesis-CA2012} % change this for oneside, 11pt, draft, etc.

% fix for pandoc 1.14
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

%--- Packages --------------------------------------------------------
\usepackage[lofdepth,lotdepth,caption=false]{subfig}
\usepackage{fancyhdr}
\usepackage{amsmath, amssymb, graphicx}
\usepackage{xspace}
\usepackage{braket}
\usepackage{color}
\usepackage{setspace}
\usepackage{fancyvrb}
\usepackage{array}
\usepackage{ifxetex,ifluatex}
\usepackage{etoolbox}

%% for the per mil symbol
\usepackage[nointegrals]{wasysym}

%% for more attractive tables
\usepackage{booktabs}
\usepackage{xcolor}
\usepackage{longtable}
\usepackage{lscape}
\usepackage{tabularx}

\usepackage[nostamp]{draftwatermark}
% % Use the following to make modification
\SetWatermarkText{DRAFT}
\SetWatermarkLightness{0.95}

% suppress bottom page numbers on 1st page of each chapt.
% because they overlap with text
%\patchcmd{\chapter}{plain}{empty}{}{} % turn off for UCD requirements of page number on every page

%---New Definitions and Commands------------------------------------------------------

\newtheorem{theorem}{Jibberish}

\bibliography{references}

\hyphenation{mar-gin-al-ia}

% from uw_template.tex

% commands and environments needed by pandoc snippets
% extracted from the output of `pandoc -s`
%% Make R markdown code chunks work

\ifxetex
  \usepackage{fontspec,xltxtra,xunicode}
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
\else
  \ifluatex
    \usepackage{fontspec}
    \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \else
    \usepackage[utf8]{inputenc}
  \fi
\fi
\DefineShortVerb[commandchars=\\\{\}]{\|}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\newenvironment{Shaded}{}{}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.56,0.13,0.00}{{#1}}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.25,0.63,0.44}{{#1}}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.25,0.63,0.44}{{#1}}}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.25,0.63,0.44}{{#1}}}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.25,0.44,0.63}{{#1}}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.25,0.44,0.63}{{#1}}}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.38,0.63,0.69}{\textit{{#1}}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{{#1}}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{1.00,0.00,0.00}{\textbf{{#1}}}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.02,0.16,0.49}{{#1}}}
\newcommand{\RegionMarkerTok}[1]{{#1}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{1.00,0.00,0.00}{\textbf{{#1}}}}
\newcommand{\NormalTok}[1]{{#1}}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}
\newcommand{\BuiltInTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.00,0.44,0.13}{\textbf{{#1}}}}

\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex,
              colorlinks=true,
              linkcolor=blue]{hyperref}
\else
  \usepackage[unicode=true,
              colorlinks=true,
              linkcolor=blue]{hyperref}
\fi
\hypersetup{breaklinks=true, pdfborder={0 0 0}}
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{0}

%---Set Margins ------------------------------------------------------
\setlength\oddsidemargin{0.25 in} \setlength\evensidemargin{0.25 in} \setlength\textwidth{6.25 in} \setlength\textheight{8.5 in} %8.5
\setlength\footskip{0.25 in} \setlength\topmargin{0 in} \setlength\headheight{0.25 in} \setlength\headsep{0.25 in}

%%%---DOCUMENT---%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

%=== Preliminary Pages ============================================
\begin{ucfrontmatter}

  %%%%%%%%%%%%%%%%%%%%%%%%%%%
  % TITLE PAGE INFORMATION % % modified to meet UCDavis
  %%%%%%%%%%%%%%%%%%%%%%%%%%%

  \title{Population genetics of a sentinel stream-breeding frog (\emph{Rana
boylii})}

  \author{Ryan A Peek}
  \prevdegreeA{B.S. (University of California, Davis) 2002}
  \prevdegreeB{M.S. (University of San Francisco) 2010}
  \report{DISSERTATION} 
  \degree{DOCTOR OF PHILOSOPHY} 
  \degreemonth{September} \degreeyear{2018}
  \chair{Michael R. Miller}  % this is your advisor
  \othermemberA{Peter B. Moyle} % This is a member of your committee
  \othermemberB{Mark W. Schwartz} % This is a member of your committee
  \othermemberC{} % This is a member of your committee
  \numberofmembers{3} % should match the number of entries above (chair + othermembers)

  \field{Ecology}
  \campus{Davis}
	
	\maketitle
	\approvalpage
	%\copyrightpage % if you want

  % DEDICATION %
  %%%%%%%%%%%%%%%%%%%%
    \begin{dedication}

      \vspace*{25ex}
      \begin{center}
      \begin{Large}

        ``\emph{One thing to remember is to talk to the animals. If you do, they
        will talk back to you. But if you don't talk to them, they won't talk
        back to you, then you won't understand. And when you don't understand,
        you will fear, and when you fear, you will destroy the animals, and if
        you destroy the animals, you will destroy yourself}''\\
        (Chief Dan George, Tseil-Waututh Nation, North Vancouver)

      \end{Large}
      \end{center}
  \end{dedication}
  % ACKNOWLEDGEMENTS %
  %%%%%%%%%%%%%%%%%%%%
  \begin{acknowledgements}
    For all the curious people who have come before and hopefully after, I
    want to acknowledge you, and I hope we can do better to inspire and
    support those voices that may not have had the opportunities or
    priviledge I have had. I am lucky to have had all I have had, and
    finishing a dissertation requires a community, and this dissertation
    would not have happened if it wasn't for the amazing community of
    family, friends, and colleagues who helped me every step of the way. In
    particular, thank you to my partner, wife and best-friend Leslie---you
    are my sun and gravity---you held me together, anchored our family, and
    made it possible to run this crazy academic ultra-marathon. To my
    dearest little tadpoles, Connor and Genevieve, you inspire me, you make
    me laugh every day, and you remind me the world still has hope as long
    as we nourish joy and curiosity. Thank you for being you, and I hope one
    day you forgive me for the amount of time I've spent staring at a
    computer. Thanks to my mom for all the support, love, and baked goods.
    Sibling, thank you for consistently inspiring me, listening to me, and
    being the best sibling one could ask for. And for all my close friends,
    bandmates, and officemates (you know who you are), you keep me sane, you
    motivate me, and you remind me every day that I really love this crazy
    journey. John, your shed and couch have probably single-handedly kept me
    anchored in ways I can't even express\ldots{}also your friendship. Thank
    you for your time, humor, and general levity. Thanks to my Dad, who has
    cajoled, pestered, and annoyed me for far too long to ``get a PhD''. . .
    thanks for believing it was possible even when I didn't. Also, please
    never suggest anything like this again. And to my committee and my
    colleagues at the Center for Watershed Sciences, you have all been an
    amazing resource in providing feedback, guidance, and support throughout
    my graduate student career. Finally, to my cohort and fellow students in
    the GGE, this has been a great place to grow and mature as a scientist
    and researcher. Thank you all.
  \end{acknowledgements}
  % removed CV section from this but see gauchodown or huskydown

  % ABSTRACT %
  %%%%%%%%%%%%%%%%%%%%%%%%%%%
  \begin{abstract}
    \addcontentsline{toc}{chapter}{Abstract}

    \emph{Rana boylii} is an imperiled frog species native to CA and OR, and
    it is currently designated as a species of special concern (CDFW) in the
    state of CA. It has been petitioned as candidate for federal (USFWS) and
    state (CDFW) listing. As a lotic breeding amphibian, \emph{R. boylii} is
    tied closely to local flow regimes in the watersheds it inhabits and is
    therefore particularly sensitive to alterations to the natural flow
    regime. Effective conservation management of this species should
    consider and prioritize maintenance of genetic diversity as part of any
    listing decision because it is closely related to the evolutionary
    capacity for adaptation to environmental changes. Conservation of
    genetic diversity in this species will require several components,
    including refining potential conservation units (i.e., distinct
    population segments) and quantifying of genetic diversity and genetic
    diversity trajectories across the species range. To assess these
    components, fine-scale and landscape-scale analyses were conducted using
    genomic data from over 600 samples from 89 localities across the range
    of the species. Six genomically-distinct groups were identified, as well
    as population subdivisions at local watershed scales. One major impact
    on \emph{R. boylii} populations has been river regulation. River
    regulation has been implicated as a cause of fundamental changes to
    downstream aquatic ecosystems. Regulation changes the natural flow
    regime which may restrict population connectivity and decrease genetic
    diversity in some species. Since population connectivity and the
    maintenance of genetic diversity are fundamental drivers of long-term
    persistence, understanding the extent that river regulation impacts
    these critical attributes of genetic health is an important goal.
    However, the extent to which \emph{R. boylii} populations in regulated
    rivers have maintained connectivity and genetic diversity is unknown.
    The impacts of river regulation on \emph{R. boylii} were investigated
    with genomic data to explore the potential for long-term persistence of
    \emph{R. boylii} under continued regulation. \emph{R. boylii} in
    regulated rivers showed striking patterns of isolation and trajectories
    of genetic diversity loss relative to unregulated rivers. For example,
    river regulation explained the greatest amount of variance in population
    genetic differentiation compared with other covariates including
    geographic distance. Importantly, patterns of connectivity and genetic
    diversity loss were observed regardless of regulation level but were
    most prominent in locations with the greatest regulation intensity.
    Using the same genomic data, fine-scale analyses of \emph{R. boylii} and
    \emph{R. sierrae} in a single region of the Sierra Nevada of California
    was conducted to evaluate the potential for hybridization between
    species. Hybridization between species may combine parental genotypes in
    ways that yield reproductively sterile or isolated lineages, and
    hybridization events may be short-lived and difficult to detect. Limited
    hybridization between the species was detected in the Feather basin,
    though it appears these are terminal events based on PCA, admixture, and
    tests of heterozygosity using species diagnostic SNPs. Finally,
    rangewide quantification and comparison of genomic variation across
    populations indicates the southern coast, southern Sierra Nevada, and
    Northern Sierra/Feather basin in California should have high
    prioritization in conservation efforts due to low genomic diversity and
    trajectories of diversity loss. More broadly, these results demonstrate
    both the critical need for regional conservation in a sentinel river
    species, and the utility and power of genetic methods for assessing and
    monitoring sensitive species across many scales.

    %\abstractsignature
  \end{abstract}
  % Table of Contents %
  %%%%%%%%%%%%%%%%%%%%%%%%%%%
	\tableofcontents

\end{ucfrontmatter} % end of the preliminary pages
\begin{ucmainmatter}

\hypertarget{uw-thesis-fields}{%
\chapter{UW thesis fields}\label{uw-thesis-fields}}

Placeholder

\hypertarget{reg-health}{%
\chapter{Flow regulation associated with decreased genetic health of a
river-breeding frog species}\label{reg-health}}

Placeholder

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

\hypertarget{methods}{%
\section{Methods}\label{methods}}

\hypertarget{sample-collection}{%
\subsection{Sample collection}\label{sample-collection}}

\hypertarget{de-novo-assembly}{%
\subsection{De novo assembly}\label{de-novo-assembly}}

\hypertarget{rapture-sequencing}{%
\subsection{Rapture sequencing}\label{rapture-sequencing}}

\hypertarget{principal-component-analysis}{%
\subsection{Principal component
analysis}\label{principal-component-analysis}}

\hypertarget{genetic-differentiation-and-diversity-estimates}{%
\subsection{Genetic differentiation and diversity
estimates}\label{genetic-differentiation-and-diversity-estimates}}

\hypertarget{boosted-regression-tree-modeling-of-variance-in-fst}{%
\subsection{Boosted regression tree modeling of variance in
FST}\label{boosted-regression-tree-modeling-of-variance-in-fst}}

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

\hypertarget{this-chunk-ensures-that-the-huskydown-package-is}{%
\chapter{This chunk ensures that the huskydown package
is}\label{this-chunk-ensures-that-the-huskydown-package-is}}

Placeholder

\hypertarget{introduction-1}{%
\section{Introduction}\label{introduction-1}}

\hypertarget{materials-and-methods}{%
\section{Materials and Methods}\label{materials-and-methods}}

\hypertarget{sampling-and-dna-extraction}{%
\subsection{Sampling and DNA
Extraction}\label{sampling-and-dna-extraction}}

\hypertarget{rapture-sequencing-1}{%
\subsection{Rapture Sequencing}\label{rapture-sequencing-1}}

\hypertarget{pca-admixture}{%
\subsection{PCA \& Admixture}\label{pca-admixture}}

\hypertarget{f1-vs-f2-test-with-species-diagnostic-snps}{%
\subsection{F1 vs F2 Test with Species Diagnostic
SNPs}\label{f1-vs-f2-test-with-species-diagnostic-snps}}

\hypertarget{demographic-modeling-with-fastsimcoal2}{%
\subsection{Demographic Modeling with
fastsimcoal2}\label{demographic-modeling-with-fastsimcoal2}}

\hypertarget{results-1}{%
\section{Results}\label{results-1}}

\hypertarget{rapture-produced-high-quality-genomic-data-for-both-r.-sierrae-and-r.-boylii}{%
\subsection{\texorpdfstring{Rapture produced high quality genomic data
for both \emph{R. sierrae} and \emph{R.
boylii}}{Rapture produced high quality genomic data for both R. sierrae and R. boylii}}\label{rapture-produced-high-quality-genomic-data-for-both-r.-sierrae-and-r.-boylii}}

\hypertarget{rangewide}{%
\chapter{\texorpdfstring{Refining conservation unit boundaries of a
sentinel stream-breeding frog (\emph{Rana boylii}) using population
genomics}{Refining conservation unit boundaries of a sentinel stream-breeding frog (Rana boylii) using population genomics}}\label{rangewide}}
\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{#include_graphics(path = "figure/uw.png")}

\CommentTok{# Here is a reference to the UW logo: Figure \textbackslash{}@ref(fig:uwlogo).  Note the use of the `fig:` code here.}
\end{Highlighting}
\end{Shaded}
\hypertarget{introduction-2}{%
\section{INTRODUCTION}\label{introduction-2}}

The use of modern genomic sequencing technology has greatly advanced the
ability for higher resolution analyses of both geographic and ecological
patterns in populations (Nunziata et al. 2017, Hendricks et al. 2018,
Barbosa et al. 2018). Reduced representation sequencing methods such as
restriction site-associated DNA sequencing (RADSeq) (Miller et al.~2007,
Baird et al.~2008, Ali et al.~2016) provides a powerful tool to address
ecological genomics questions at scales that were previously impossible
using traditional field methods. Furthermore, new methods such as RAD
Capture (Rapture) (Ali et al.~2016) adapt RADSeq to target desired loci
and allow highly efficient genotyping of thousands of individuals at
once. As historical and future landscape use can influence species
demography and migration patterns (Burkey 1989, Anderson and Beer 2009,
Barbosa et al.~2018), these genomic tools will be invaluable for
assessing critical factors for long-term persistence in sensitive
populations or species. The ecological integrity of freshwater systems
and their constituent biota are rapidly declining globally (Ricciardi
and Rasmussen 1999), and conservation efforts will require assessment of
the adaptive capacity of populations to rapid environmental change.
Given limited capacity to conserve, it is important to define and
establish clear geographic boundaries for conservation units such as
distinct population segments across a species' range. Delineation of
distinct population segments can be used for prioritizing objectives in
conservation management. Furthermore, quantification and comparison of
relative genetic diversity within and among populations can provide
additional information as a benchmark for future assessment responses to
conservation actions. Thus, quantifying and linking landscape change
with genetic diversity metrics may provide an important baseline to
track how sensitive populations respond to future environmental change
(through reduced adaptive potential) as well as evaluating whether
restoration efforts are effective (i.e., increasing genetic
connectivity, diversity, effective breeder/population size).\\
Amphibians are particularly sensitive to changes in the ecosystem due to
their physiology and ontogeny (Davidson et al.~2002, Beebee and
Griffiths 2005), thus the ability to utilize environmental variables as
life history cues can be especially important. In highly dynamic
riverine environments, organisms must constantly adapt to temporal and
spatial changes. One such sentinel stream-breeding species is the
Foothill yellow-legged frog (Rana boylii), a native to California and
Oregon which historically occurred in lower elevation (0-1500m) streams
and rivers from Southern Oregon to northern Baja California west of the
Sierra-Cascade crest (Stebbins 2003). As a lotic breeding amphibian, R.
boylii is tied closely to the local hydrology in watersheds it inhabits,
and therefore it is particularly sensitive to alterations to flow
regimes (Kupferberg 1996, Lind et al.~1996, Kupferberg et al.~2012). As
with many amphibians in California (Davidson 2004, Vredenburg and Wake
2007, Peek 2010, Kupferberg et al.~2012, Thomson et al.~2016), there
have been significant population declines across the former range of
this species, particularly in southern California and the Sierra Nevada
where it has been extirpated from approximately 50 percent of its
historical range (Jennings and Hayes 1994, Davidson et al.~2002). Rana
boylii, currently designated as a species of special concern (CDFW) in
the state of CA, has been petitioned as candidate for listing under the
federal (USFWS) Endangered Species Act (USFWS 2014) as well as the state
(CDFW) Endangered Species Act. Effective conservation management of this
species will need to consider and prioritize maintenance of genetic
diversity as part of any listing decision because it is closely related
to the evolutionary capacity for adaptation to environmental changes
(Lande and Shannon 1996) . Thus, utilizing genetic data provides a
potentially informative process for identifying the impacts of
anthropogenic and environmental change on the process of adaptation.
Establishing high-resolution genetic boundaries for populations across
the species range as well as delineation of distinct population segments
that can be more effective in conservation management when coupled with
quantification of relative genomic diversity metrics (i.e., genomic
diversity, population connectivity). A recent study by McCartney-Melstad
et al. (2018) identified five major clades in R. boylii with strong
geographically structured genetic subdivision across its range in
California and Oregon. Here we provide an additional population genomic
analysis across the range of this declining sentinel stream species that
is currently a candidate for listing. We provide additional geographic
and genetic resolution to McCartney-Melstad et al. (2018), as well as
quantify genetic diversity metrics across subpopulations and clades as
both a reference and assessment of the potential for long-term
persistence across this species' range.

\appendix

\hypertarget{the-first-appendix}{%
\chapter{The First Appendix}\label{the-first-appendix}}

This first appendix includes all of the R chunks of code that were
hidden throughout the document (using the \texttt{include\ =\ FALSE}
chunk tag) to help with readibility and/or setup.

\textbf{In the main Rmd file}

\textbf{In Chapter \ref{ref-labels}:}

\hypertarget{the-second-appendix-for-fun}{%
\chapter{The Second Appendix, for
Fun}\label{the-second-appendix-for-fun}}

\hypertarget{colophon}{%
\chapter*{Colophon}\label{colophon}}
\addcontentsline{toc}{chapter}{Colophon}

This document is set in \href{https://github.com/georgd/EB-Garamond}{EB
Garamond}, \href{https://github.com/adobe-fonts/source-code-pro/}{Source
Code Pro} and \href{http://www.latofonts.com/lato-free-fonts/}{Lato}.
The body text is set at 11pt with \(\familydefault\).

It was written in R Markdown and \(\LaTeX\), and rendered into PDF using
\href{https://github.com/benmarwick/huskydown}{huskydown} and
\href{https://github.com/rstudio/bookdown}{bookdown}.

This document was typeset using the XeTeX typesetting system, and the
\href{http://staff.washington.edu/fox/tex/}{University of Washington
Thesis class} class created by Jim Fox. Under the hood, the
\href{https://github.com/UWIT-IAM/UWThesis}{University of Washington
Thesis LaTeX template} is used to ensure that documents conform
precisely to submission standards. Other elements of the document
formatting source code have been taken from the
\href{https://github.com/stevenpollack/ucbthesis}{Latex, Knitr, and
RMarkdown templates for UC Berkeley's graduate thesis}, and
\href{https://github.com/suchow/Dissertate}{Dissertate: a LaTeX
dissertation template to support the production and typesetting of a PhD
dissertation at Harvard, Princeton, and NYU}

The source files for this thesis, along with all the data files, have
been organised into an R package, xxx, which is available at
\url{https://github.com/xxx/xxx}. A hard copy of the thesis can be found
in the University of Washington library.

This version of the thesis was generated on 2018-09-10 15:59:58. The
repository is currently at this commit:

The computational environment that was used to generate this version is
as follows:
\begin{verbatim}
Session info -------------------------------------------------------------
\end{verbatim}
\begin{verbatim}
 setting  value                       
 version  R version 3.5.1 (2018-07-02)
 system   x86_64, darwin15.6.0        
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2018-09-10                  
\end{verbatim}
\begin{verbatim}
Packages -----------------------------------------------------------------
\end{verbatim}
\begin{verbatim}
 package     * version    date       source                               
 assertthat    0.2.0      2017-04-11 CRAN (R 3.5.0)                       
 backports     1.1.2      2017-12-13 CRAN (R 3.5.0)                       
 base        * 3.5.1      2018-07-05 local                                
 bindr         0.1.1      2018-03-13 CRAN (R 3.5.0)                       
 bindrcpp      0.2.2      2018-03-29 CRAN (R 3.5.0)                       
 bookdown      0.7        2018-02-18 CRAN (R 3.5.0)                       
 colorspace    1.3-2      2016-12-14 CRAN (R 3.5.0)                       
 compiler      3.5.1      2018-07-05 local                                
 crayon        1.3.4      2017-09-16 CRAN (R 3.5.0)                       
 datasets    * 3.5.1      2018-07-05 local                                
 devtools    * 1.13.6     2018-06-27 CRAN (R 3.5.0)                       
 digest        0.6.16     2018-08-22 CRAN (R 3.5.1)                       
 dplyr       * 0.7.6      2018-06-29 CRAN (R 3.5.0)                       
 evaluate      0.11       2018-07-17 CRAN (R 3.5.0)                       
 ggplot2     * 3.0.0.9000 2018-09-04 Github (tidyverse/ggplot2@6e545dc)   
 git2r         0.23.0     2018-07-17 CRAN (R 3.5.0)                       
 glue          1.3.0      2018-07-17 CRAN (R 3.5.0)                       
 graphics    * 3.5.1      2018-07-05 local                                
 grDevices   * 3.5.1      2018-07-05 local                                
 grid          3.5.1      2018-07-05 local                                
 gtable        0.2.0      2016-02-26 CRAN (R 3.5.0)                       
 hms           0.4.2      2018-03-10 CRAN (R 3.5.0)                       
 htmltools     0.3.6      2017-04-28 CRAN (R 3.5.0)                       
 httr          1.3.1      2017-08-20 CRAN (R 3.5.0)                       
 huskydown     0.0.5      2018-09-04 Github (benmarwick/huskydown@3ef00c9)
 kableExtra  * 0.9.0      2018-05-21 CRAN (R 3.5.0)                       
 knitr       * 1.20       2018-02-20 CRAN (R 3.5.0)                       
 lazyeval      0.2.1      2017-10-29 CRAN (R 3.5.0)                       
 magrittr      1.5        2014-11-22 CRAN (R 3.5.0)                       
 memoise       1.1.0      2017-04-21 CRAN (R 3.5.0)                       
 methods     * 3.5.1      2018-07-05 local                                
 munsell       0.5.0      2018-06-12 CRAN (R 3.5.0)                       
 pillar        1.3.0      2018-07-14 CRAN (R 3.5.0)                       
 pkgconfig     2.0.2      2018-08-16 CRAN (R 3.5.0)                       
 plyr          1.8.4      2016-06-08 CRAN (R 3.5.0)                       
 purrr         0.2.5      2018-05-29 CRAN (R 3.5.0)                       
 R6            2.2.2      2017-06-17 CRAN (R 3.5.0)                       
 Rcpp          0.12.18    2018-07-23 CRAN (R 3.5.1)                       
 readr       * 1.1.1      2017-05-16 CRAN (R 3.5.0)                       
 rlang         0.2.2      2018-08-16 CRAN (R 3.5.0)                       
 rmarkdown     1.10       2018-06-11 cran (@1.10)                         
 rprojroot     1.3-2      2018-01-03 CRAN (R 3.5.0)                       
 rstudioapi    0.7        2017-09-07 CRAN (R 3.5.0)                       
 rvest         0.3.2      2016-06-17 CRAN (R 3.5.0)                       
 scales        1.0.0.9000 2018-08-29 Github (hadley/scales@0f7a186)       
 stats       * 3.5.1      2018-07-05 local                                
 stringi       1.2.4      2018-07-20 CRAN (R 3.5.0)                       
 stringr       1.3.1      2018-05-10 CRAN (R 3.5.0)                       
 tibble        1.4.2      2018-01-22 CRAN (R 3.5.0)                       
 tidyselect    0.2.4      2018-02-26 CRAN (R 3.5.0)                       
 tools         3.5.1      2018-07-05 local                                
 utils       * 3.5.1      2018-07-05 local                                
 viridisLite   0.3.0      2018-02-01 CRAN (R 3.5.0)                       
 withr         2.1.2      2018-08-29 Github (jimhester/withr@8b9cee2)     
 xfun          0.3        2018-07-06 CRAN (R 3.5.0)                       
 xml2          1.2.0      2018-01-24 CRAN (R 3.5.0)                       
 yaml          2.2.0      2018-07-25 CRAN (R 3.5.0)                       
\end{verbatim}
\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

Placeholder

\hypertarget{refs}{}
\leavevmode\hypertarget{ref-barbosa_integrative_2018}{}%
Barbosa, S., F. Mestre, T. A. White, J. Paupério, P. C. Alves, and J. B.
Searle. 2018. Integrative approaches to guide conservation decisions:
Using genomics to define conservation units and functional corridors.
Molecular ecology.

\leavevmode\hypertarget{ref-hendricks_recent_2018}{}%
Hendricks, S., E. C. Anderson, T. Antao, L. Bernatchez, B. R. Forester,
B. Garner, B. K. Hand, P. A. Hohenlohe, M. Kardos, B. Koop, A.
Sethuraman, R. S. Waples, and G. Luikart. 2018. Recent advances in
conservation and population genomics data analysis. Evolutionary
applications.

\leavevmode\hypertarget{ref-nunziata_genomic_2017}{}%
Nunziata, S. O., S. L. Lance, D. E. Scott, E. M. Lemmon, and D. W.
Weisrock. 2017. Genomic data detect corresponding signatures of
population size change on an ecological time scale in two salamander
species. Molecular ecology 26:1060--1074.

\end{ucmainmatter}
\end{document}

%---Set Headers and Footers ------------------------------------------------------
\pagestyle{fancy}
% UCD
%\renewcommand{\chaptermark}[1]{\markboth{{\sf #1 \hspace*{\fill} Chapter~}}{} }
% GAUCHO
\fancyhf{}
\renewcommand{\chaptermark}[1]{\markboth{{\sf #1 \hspace*{\fill} Chapter~\thechapter}}{} }
\renewcommand{\sectionmark}[1]{\markright{ {\sf Section~\thesection 
\hspace*{\fill} #1 }}}


\makeatletter \if@twoside \fancyhead[LO]{\small \rightmark} \fancyhead[RE]{\small\leftmark} \else \fancyhead[LO]{\small\leftmark}
\fancyhead[RE]{\small\rightmark} \fi

\def\cleardoublepage{\clearpage\if@openright \ifodd\c@page\else
  \hbox{}
  \vspace*{\fill}
  \begin{center}
    This page intentionally left blank
  \end{center}
  \vspace{\fill}
  \thispagestyle{plain}
  \newpage
  \fi \fi}
\makeatother
\fancyfoot[c]{\textrm{\textup{\thepage}}} % page number
\fancyfoot[C]{\thepage}
\renewcommand{\headrulewidth}{0.4pt}

\fancypagestyle{plain} { \fancyhf{} \fancyfoot[C]{\thepage}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}}
